{"nbformat":4,"nbformat_minor":0,"metadata":{"colab":{"provenance":[{"file_id":"1o8r4cyIbXnm8LMTTtbcw9GbWp2ZcFK47","timestamp":1712916222915}],"authorship_tag":"ABX9TyMM11pupVLSTvrPZlzaqEyT"},"kernelspec":{"name":"python3","display_name":"Python 3"},"language_info":{"name":"python"}},"cells":[{"cell_type":"code","execution_count":1,"metadata":{"id":"3um1TBoGLVb0","executionInfo":{"status":"ok","timestamp":1712916284669,"user_tz":-120,"elapsed":1153,"user":{"displayName":"Anna Kiepura","userId":"06512247374108941998"}}},"outputs":[],"source":["import json\n","import pickle"]},{"cell_type":"code","source":["from google.colab import drive\n","drive.mount('/content/drive', force_remount=True)"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"er4II595LYNe","executionInfo":{"status":"ok","timestamp":1712916304519,"user_tz":-120,"elapsed":19854,"user":{"displayName":"Anna Kiepura","userId":"06512247374108941998"}},"outputId":"0ec974c9-2a16-46c7-8a99-57147b8cc174"},"execution_count":2,"outputs":[{"output_type":"stream","name":"stdout","text":["Mounted at /content/drive\n"]}]},{"cell_type":"code","source":["%cd /content/drive/My Drive/SciPara_2024_release"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"PQj3fuI-LYQE","executionInfo":{"status":"ok","timestamp":1712916304520,"user_tz":-120,"elapsed":6,"user":{"displayName":"Anna Kiepura","userId":"06512247374108941998"}},"outputId":"e2bc8fa6-767c-484f-d54e-71506ade2cae"},"execution_count":3,"outputs":[{"output_type":"stream","name":"stdout","text":["/content/drive/My Drive/SciPara_2024_release\n"]}]},{"cell_type":"markdown","source":["# Load the dataset"],"metadata":{"id":"caTo94MnLl0r"}},{"cell_type":"code","source":["with open('base_dataset_abstract_previous.pkl', 'rb') as f:\n","    loaded_dict = pickle.load(f)"],"metadata":{"id":"grKqPvBHLYTM","executionInfo":{"status":"ok","timestamp":1712916308217,"user_tz":-120,"elapsed":688,"user":{"displayName":"Anna Kiepura","userId":"06512247374108941998"}}},"execution_count":4,"outputs":[]},{"cell_type":"markdown","source":["In the key_list, we have the paper ids of the papers in our dataset"],"metadata":{"id":"YSPs17iSMczb"}},{"cell_type":"code","source":["key_list = list(loaded_dict.keys())"],"metadata":{"id":"1wJOYn7FLYVq","executionInfo":{"status":"ok","timestamp":1712916308763,"user_tz":-120,"elapsed":4,"user":{"displayName":"Anna Kiepura","userId":"06512247374108941998"}}},"execution_count":5,"outputs":[]},{"cell_type":"code","source":["key_list[0]"],"metadata":{"colab":{"base_uri":"https://localhost:8080/","height":35},"id":"xfft2P6SLYYI","executionInfo":{"status":"ok","timestamp":1712916309038,"user_tz":-120,"elapsed":3,"user":{"displayName":"Anna Kiepura","userId":"06512247374108941998"}},"outputId":"004444f4-cdbe-4bfd-8f74-7bdf7fe17262"},"execution_count":6,"outputs":[{"output_type":"execute_result","data":{"text/plain":["'paper_1'"],"application/vnd.google.colaboratory.intrinsic+json":{"type":"string"}},"metadata":{},"execution_count":6}]},{"cell_type":"markdown","source":["For each paper, we have the paper's body (divided into paragraphs) and abstract"],"metadata":{"id":"xJAd5NRHNyHD"}},{"cell_type":"code","source":["loaded_dict[\"paper_1\"].keys()"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"fdeO8hhnN2mM","executionInfo":{"status":"ok","timestamp":1712916310081,"user_tz":-120,"elapsed":2,"user":{"displayName":"Anna Kiepura","userId":"06512247374108941998"}},"outputId":"a1017965-a1d3-4166-f937-62dfffd08861"},"execution_count":7,"outputs":[{"output_type":"execute_result","data":{"text/plain":["dict_keys(['paper', 'abstract'])"]},"metadata":{},"execution_count":7}]},{"cell_type":"code","source":["loaded_dict[\"paper_1\"]['paper']"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"k0cfFRfZLYaY","executionInfo":{"status":"ok","timestamp":1712916310694,"user_tz":-120,"elapsed":12,"user":{"displayName":"Anna Kiepura","userId":"06512247374108941998"}},"outputId":"a4f91705-01ba-4e32-be31-67016ab73ea3"},"execution_count":8,"outputs":[{"output_type":"execute_result","data":{"text/plain":["[{'id': 36211,\n","  'text': 'SKIP\\nPaper title: Phylogenetic analyses provide the first insights into the evolution of OVATE family proteins in land plants. \\n\\n(#1) The OVATE gene was first identified as an important regulator of fruit shape in tomato, in which a naturally occurring premature stop codon in OVATE results in pear-shaped fruit with longitudinal elongation and neck constriction (Liu et al., 2002) .\\n(#2) This revealed a previously uncharacterized class of regulatory genes in plant development, which encode proteins with a conserved 70 amino acid C-terminal domain.\\n(#3) This domain was designated as the OVATE domain, also known as DUF623 (Domain of Unknown Function 623), and proteins containing this domain were designated OVATE family proteins (OFPs), which are found exclusively in plants (Hackbusch et al., 2005; Wang et al., 2007 Wang et al., , 2011 .',\n","  'label': [[0, 4, 'Bad structure'],\n","   [129, 132, 'Topic sentence'],\n","   [129, 133, 'Score 3'],\n","   [138, 143, 'Paragraph Topic'],\n","   [186, 204, 'Topic attribute 1'],\n","   [277, 282, 'Paragraph Topic'],\n","   [384, 388, 'Score 4'],\n","   [384, 388, 'Supporting detail sentence'],\n","   [552, 556, 'Score 2'],\n","   [552, 556, 'Supporting detail sentence'],\n","   [591, 596, 'Paragraph Topic'],\n","   [712, 717, 'Paragraph Topic'],\n","   [712, 740, 'Topic attribute 2']]},\n"," {'id': 36212,\n","  'text': 'SKIP\\n(#1) To date, OFPs have been primarily characterized in arabidopsis (AtOFPs) and demonstrated to regulate plant growth and development (Hackbusch et al., 2005; Pagnussat et al., 2007; Wang et al., 2007 Wang et al., , 2010 Li et al., 2011) .\\n(#2) AtOFPs were shown to have close functional interactions with three amino acid loop extension (TALE) homeodomain proteins, and AtOFP1 and AtOFP5 regulate the sub-cellular localization of TALE homeoproteins (Hackbusch et al., 2005) .\\n(#3) The plant TALE proteins are a conserved superclass of homeodomain proteins characterized by an extension of three amino acids between helices 1 and 2 of the homeodomain (Bertolino et al., 1995) , and comprise two sub-classes called the KNOTTED-like homeobox (KNOX) and BEL1-like homeodomain (BELL) proteins Tsiantis, 2009, 2010; Hamant and Pautot, 2010) .\\n(#4) Furthermore, it has been well documented that interactions between KNOX and BELL proteins result in heterodimers regulating plant development in a connected and complex network (Bellaoui et al., 2001; Smith et al., 2002; Smith and Hake, 2003; Chen et al., 2004; Hackbusch et al., 2005; Cole et al., 2006) .\\n(#5) AtOFP1 has been reported to function as an active transcriptional repressor of AtGA20ox1 in the gibberellin (GA) biosynthesis pathway, suppressing cell elongation (Wang et al., 2007) .\\n(#6) A recent study also indicated that AtOFP1 interacts with AtKu70, a protein involved in DNA repair through the non-homologous end-joining pathway .\\n(#7) Similar to AtOFP1, AtOFP4 acts as a transcriptional repressor and has been proposed to form a functional complex with KNAT7, one of four class II arabidopsis KNOTTED1-like Arabidopsis thaliana (KNAT) members (Hake et al., 2004; Hamant and Pautot, 2010) , in regulating secondary cell wall formation (Li et al., 2011) .\\n(#8) AtOFP5, reported to interact with both BLH1 and KNAT3, which belong to BELL and KNOX sub-classes of TALE homeodomain proteins, respectively (Hackbusch et al., 2005) , can act as regulator of the BELL-KNOX TALE complex involved in normal embryo sac development in arabidopsis (Pagnussat et al., 2007) .\\n(#9) Recently, a genome-wide analysis of AtOFPs revealed conserved functions as transcriptional repressors, with overexpression leading to a number of abnormal phenotypes, implying novel roles in regulating plant growth and development .',\n","  'label': [[0, 4, 'Bad structure'],\n","   [5, 9, 'Score 4'],\n","   [5, 9, 'Topic sentence'],\n","   [19, 23, 'Paragraph Topic'],\n","   [74, 80, 'Paragraph Topic'],\n","   [246, 250, 'Score 2'],\n","   [246, 250, 'Supporting detail sentence'],\n","   [251, 257, 'Paragraph Topic'],\n","   [345, 349, 'Topic attribute 1'],\n","   [377, 383, 'Topic attribute 2'],\n","   [388, 394, 'Topic attribute 4'],\n","   [436, 441, 'Topic attribute 1'],\n","   [483, 487, 'Score 3'],\n","   [483, 487, 'Supporting detail sentence'],\n","   [498, 503, 'Topic attribute 1'],\n","   [747, 751, 'Topic attribute 3'],\n","   [757, 785, 'Topic attribute 6'],\n","   [844, 848, 'Supporting detail sentence'],\n","   [845, 848, 'Score 4'],\n","   [916, 920, 'Topic attribute 3'],\n","   [925, 938, 'Topic attribute 6'],\n","   [1156, 1160, 'Supporting detail sentence'],\n","   [1157, 1160, 'Score 3'],\n","   [1161, 1167, 'Topic attribute 2'],\n","   [1346, 1350, 'Score 4'],\n","   [1346, 1350, 'Supporting detail sentence'],\n","   [1386, 1392, 'Topic attribute 2'],\n","   [1498, 1502, 'Score 2'],\n","   [1498, 1502, 'Supporting detail sentence'],\n","   [1514, 1520, 'Topic attribute 2'],\n","   [1621, 1626, 'Topic attribute 5'],\n","   [1697, 1701, 'Topic attribute 5'],\n","   [1822, 1826, 'Score 3'],\n","   [1822, 1826, 'Supporting detail sentence'],\n","   [1827, 1833, 'Topic attribute 4'],\n","   [1875, 1880, 'Topic attribute 5'],\n","   [1898, 1902, 'Topic attribute 6'],\n","   [1907, 1911, 'Topic attribute 3'],\n","   [1927, 1931, 'Topic attribute 1'],\n","   [2022, 2026, 'Topic attribute 6'],\n","   [2027, 2031, 'Topic attribute 3'],\n","   [2032, 2036, 'Topic attribute 1'],\n","   [2129, 2133, 'Score 3'],\n","   [2129, 2133, 'Supporting detail sentence'],\n","   [2170, 2176, 'Paragraph Topic']],\n","  'previous_para': 'SKIP\\nPaper title: Phylogenetic analyses provide the first insights into the evolution of OVATE family proteins in land plants. \\n\\n(#1) The OVATE gene was first identified as an important regulator of fruit shape in tomato, in which a naturally occurring premature stop codon in OVATE results in pear-shaped fruit with longitudinal elongation and neck constriction (Liu et al., 2002) .\\n(#2) This revealed a previously uncharacterized class of regulatory genes in plant development, which encode proteins with a conserved 70 amino acid C-terminal domain.\\n(#3) This domain was designated as the OVATE domain, also known as DUF623 (Domain of Unknown Function 623), and proteins containing this domain were designated OVATE family proteins (OFPs), which are found exclusively in plants (Hackbusch et al., 2005; Wang et al., 2007 Wang et al., , 2011 .'},\n"," {'id': 36213,\n","  'text': 'SKIP\\n(#1) Gene homologues containing the conserved OVATE domain have been found in tomato, arabidopsis and rice (Liu et al., 2002) .\\n(#2) Furthermore, OVATE-like genes appear to be conserved in many plant species, with conserved genomic microsynteny discovered not only between Solanaceae relatives , but also in distantly related species (arabidopsis, snapdragon, papaya, poplar, grape and coffee tree) (Causier et al., 2010; Guyot et al., 2012) , suggesting ancestral synteny of these regions in plants.\\n(#3) In pepper (Capsicum annuum), a relative of tomato in the Solanaceae family, an OVATE family member (CaOvate) was also shown to be involved in determining fruit shape by negatively affecting the expression of CaGA20ox1 (Tsaballa et al., 2011) .',\n","  'label': [[0, 4, 'Bad structure'],\n","   [5, 8, 'Topic sentence'],\n","   [5, 9, 'Score 4'],\n","   [10, 63, 'Paragraph Topic'],\n","   [83, 89, 'Topic attribute 1'],\n","   [133, 137, 'Score 3'],\n","   [134, 137, 'Supporting detail sentence'],\n","   [151, 167, 'Paragraph Topic'],\n","   [506, 510, 'Score 3'],\n","   [506, 510, 'Supporting detail sentence'],\n","   [554, 560, 'Topic attribute 1'],\n","   [590, 609, 'Paragraph Topic']],\n","  'previous_para': 'SKIP\\n(#1) To date, OFPs have been primarily characterized in arabidopsis (AtOFPs) and demonstrated to regulate plant growth and development (Hackbusch et al., 2005; Pagnussat et al., 2007; Wang et al., 2007 Wang et al., , 2010 Li et al., 2011) .\\n(#2) AtOFPs were shown to have close functional interactions with three amino acid loop extension (TALE) homeodomain proteins, and AtOFP1 and AtOFP5 regulate the sub-cellular localization of TALE homeoproteins (Hackbusch et al., 2005) .\\n(#3) The plant TALE proteins are a conserved superclass of homeodomain proteins characterized by an extension of three amino acids between helices 1 and 2 of the homeodomain (Bertolino et al., 1995) , and comprise two sub-classes called the KNOTTED-like homeobox (KNOX) and BEL1-like homeodomain (BELL) proteins Tsiantis, 2009, 2010; Hamant and Pautot, 2010) .\\n(#4) Furthermore, it has been well documented that interactions between KNOX and BELL proteins result in heterodimers regulating plant development in a connected and complex network (Bellaoui et al., 2001; Smith et al., 2002; Smith and Hake, 2003; Chen et al., 2004; Hackbusch et al., 2005; Cole et al., 2006) .\\n(#5) AtOFP1 has been reported to function as an active transcriptional repressor of AtGA20ox1 in the gibberellin (GA) biosynthesis pathway, suppressing cell elongation (Wang et al., 2007) .\\n(#6) A recent study also indicated that AtOFP1 interacts with AtKu70, a protein involved in DNA repair through the non-homologous end-joining pathway .\\n(#7) Similar to AtOFP1, AtOFP4 acts as a transcriptional repressor and has been proposed to form a functional complex with KNAT7, one of four class II arabidopsis KNOTTED1-like Arabidopsis thaliana (KNAT) members (Hake et al., 2004; Hamant and Pautot, 2010) , in regulating secondary cell wall formation (Li et al., 2011) .\\n(#8) AtOFP5, reported to interact with both BLH1 and KNAT3, which belong to BELL and KNOX sub-classes of TALE homeodomain proteins, respectively (Hackbusch et al., 2005) , can act as regulator of the BELL-KNOX TALE complex involved in normal embryo sac development in arabidopsis (Pagnussat et al., 2007) .\\n(#9) Recently, a genome-wide analysis of AtOFPs revealed conserved functions as transcriptional repressors, with overexpression leading to a number of abnormal phenotypes, implying novel roles in regulating plant growth and development .'},\n"," {'id': 36214,\n","  'text': 'SKIP\\n(#1) Despite the aforementioned analyses of OFPs, our knowledge concerning this protein family remains relatively poor.\\n(#2) In this study, we focused on the evolution of OFPs in land plants by performing, for the first time, a genome-wide comparative analysis of sequences encoding putative OFPs in a wide variety of plant genomes.\\n(#3) Furthermore, we provided a detailed understanding of the evolutionary framework of the OVATE family in the model species arabidopsis, rice and tomato, including investigation of the expression patterns of SlOFP genes.',\n","  'label': [[0, 4, 'Bad structure'],\n","   [5, 9, 'Score 4'],\n","   [5, 9, 'Transition Sentence'],\n","   [49, 53, 'Paragraph Topic'],\n","   [80, 98, 'Paragraph Topic'],\n","   [125, 129, 'Score 4'],\n","   [125, 129, 'Topic sentence'],\n","   [176, 179, 'Paragraph Topic'],\n","   [297, 301, 'Paragraph Topic'],\n","   [338, 342, 'Topic sentence'],\n","   [339, 342, 'Score 4'],\n","   [430, 442, 'Paragraph Topic']],\n","  'previous_para': 'SKIP\\n(#1) Gene homologues containing the conserved OVATE domain have been found in tomato, arabidopsis and rice (Liu et al., 2002) .\\n(#2) Furthermore, OVATE-like genes appear to be conserved in many plant species, with conserved genomic microsynteny discovered not only between Solanaceae relatives , but also in distantly related species (arabidopsis, snapdragon, papaya, poplar, grape and coffee tree) (Causier et al., 2010; Guyot et al., 2012) , suggesting ancestral synteny of these regions in plants.\\n(#3) In pepper (Capsicum annuum), a relative of tomato in the Solanaceae family, an OVATE family member (CaOvate) was also shown to be involved in determining fruit shape by negatively affecting the expression of CaGA20ox1 (Tsaballa et al., 2011) .'},\n"," {'id': 36216,\n","  'text': 'SKIP\\n(#1) From all searches, two previously unpublished AtOFPs were identified: At2g36026 (designated AtOFP19) was identified from a TBLASTN search against the TAIR (The Arabidopsis Information Resource 10.0) database (http:\\\\/\\\\/www.arabidopsis.\\n(#2) org\\\\/index.jsp) using SlOFP1 as the query; while At1g06923 (designated AtOFP20) was detected in a segmental duplication block by the PGDD (Plant Genome Duplication Database, http:\\\\/\\\\/chibba.agtec.uga.edu\\\\/duplication\\\\/) as a putative paralogue of At2g30395 (AtOFP17; see below for details).\\n(#3) All search results were collated and manually curated to produce a nonredundant data set that was then subjected to a CDD (Conserved Domain Database) search (http:\\\\/\\\\/www.ncbi.nlm.nih.gov\\\\/ Structure\\\\/cdd\\\\/cdd.shtml; Marchler-Baueret al., 2011) for the verification of the OVATE domain (PF04844).\\n(#4) All sequences were thus verified, with the exception of SlOFP16, as well as AtOFP17, AtOFP20 and their orthologues.\\n(#5) SlOFP16 no longer seems to possess the OVATE domain, whereas the AtOFP17\\\\/ 20-like proteins have a divergent but still recognizable OVATE domain in the multiple alignment with other OVATE proteins (Supplementary Data File S2).\\n(#6) Excluding SlOFP16, a total of 265 putative OVATE protein sequences were obtained from the 13 selected plant genomes.\\n(#7) The nomenclature of all the putative OVATE proteins is listed in Supplementary Data Table S2 .',\n","  'label': [[0, 4, 'Bad parse'],\n","   [5, 9, 'Score 1'],\n","   [6, 8, 'Topic sentence'],\n","   [29, 62, 'Paragraph Topic'],\n","   [79, 110, 'Topic attribute 1'],\n","   [243, 247, 'Score 1'],\n","   [290, 326, 'Topic attribute 2'],\n","   [501, 508, 'Link 1'],\n","   [534, 538, 'Score 3'],\n","   [534, 538, 'Supporting detail sentence'],\n","   [803, 830, 'Topic attribute 3'],\n","   [832, 835, 'Score 3'],\n","   [832, 835, 'Supporting detail sentence'],\n","   [892, 899, 'Link 2'],\n","   [912, 919, 'Link 1'],\n","   [953, 955, 'Score 3'],\n","   [953, 955, 'Supporting detail sentence'],\n","   [957, 964, 'Link 2'],\n","   [1022, 1029, 'Link 1'],\n","   [1088, 1100, 'Topic attribute 3'],\n","   [1138, 1152, 'Topic attribute 4'],\n","   [1184, 1186, 'Conclusion sentence'],\n","   [1184, 1186, 'Score 4'],\n","   [1230, 1244, 'Topic attribute 4'],\n","   [1306, 1308, 'Score 4'],\n","   [1306, 1308, 'Supporting detail sentence'],\n","   [1347, 1361, 'Topic attribute 4']],\n","  'previous_para': 'SKIP\\n(#1) Despite the aforementioned analyses of OFPs, our knowledge concerning this protein family remains relatively poor.\\n(#2) In this study, we focused on the evolution of OFPs in land plants by performing, for the first time, a genome-wide comparative analysis of sequences encoding putative OFPs in a wide variety of plant genomes.\\n(#3) Furthermore, we provided a detailed understanding of the evolutionary framework of the OVATE family in the model species arabidopsis, rice and tomato, including investigation of the expression patterns of SlOFP genes.'},\n"," {'id': 36217,\n","  'text': 'SKIP\\n(#1) Multiple sequence alignment of all 265 full-length putative OVATE protein sequences was performed using MUSCLE (http:\\\\/\\\\/www.ebi.ac.uk\\\\/Tools\\\\/msa\\\\/muscle\\\\/; Edgar, 2004) .\\n(#2) PtOFP5 and ZmOFP42 were re-annotated because of an insertion of 59 amino acids in the conserved OVATE domain (see Supplementary Data File S1).\\n(#3) Given the high degree of diversity among the full-length OVATE protein sequences, phylogenetic analysis was conducted on the alignment of the conserved OVATE domain.\\n(#4) Alignments were manually revised in BioEdit (Hall, 1999) , and the resulting alignment of 86 amino acid sites was used for maximum likelihood (ML) analysis using RAxML 7.0.4 (Stamatakis, 2006) , with the BLOSUM62 amino acid substitution matrix and CAT approximation, and 200 bootstrap replicates.\\n(#5) The phylogenetic relationships of OVATE proteins in arabidopsis, rice and tomato were also estimated separately using MEGA 4.0 (Kumar et al., 2008) with the Neighbor -Joining (NJ) method based on the p-distance model, pairwise deletion and 1000 bootstraps.\\n(#6) Trees were visually inspected and edited in Dendroscope (Huson et al., 2007) .\\n(#7) Since there was no prior information on placement of the root, trees were rooted with midpoint rooting.\\n(#8) The MEME (version 4.9.0) motif search tool (http:\\\\/\\\\/meme.nbcr.net\\\\/meme\\\\/ intro.html; Bailey et al., 2009) was used to detect conserved motifs within related OFPs with the following parameters: maximum number of motifs: 10; 4 ≤ motif width ≤ 70; 2 ≤ number of sites ≤ 300.\\n(#9) The sequence logos and E-values for each motif are indicated in Supplementary Data Fig.\\n(#10) S1 .',\n","  'label': [[0, 4, 'Bad parse'],\n","   [0, 4, 'Bad structure'],\n","   [6, 8, 'Supporting detail sentence'],\n","   [6, 9, 'Score 4'],\n","   [178, 180, 'Score 3'],\n","   [178, 180, 'Supporting detail sentence'],\n","   [326, 328, 'Score 3'],\n","   [326, 328, 'Supporting detail sentence'],\n","   [497, 499, 'Score 3'],\n","   [497, 499, 'Supporting detail sentence'],\n","   [799, 801, 'Score 3'],\n","   [799, 801, 'Supporting detail sentence'],\n","   [1061, 1063, 'Supporting detail sentence'],\n","   [1061, 1064, 'Score 3'],\n","   [1145, 1147, 'Score 3'],\n","   [1145, 1147, 'Supporting detail sentence'],\n","   [1254, 1256, 'Score 3'],\n","   [1254, 1256, 'Supporting detail sentence'],\n","   [1529, 1531, 'Score 3'],\n","   [1529, 1531, 'Supporting detail sentence'],\n","   [1622, 1625, 'Score 1']],\n","  'previous_para': 'SKIP\\n(#1) From all searches, two previously unpublished AtOFPs were identified: At2g36026 (designated AtOFP19) was identified from a TBLASTN search against the TAIR (The Arabidopsis Information Resource 10.0) database (http:\\\\/\\\\/www.arabidopsis.\\n(#2) org\\\\/index.jsp) using SlOFP1 as the query; while At1g06923 (designated AtOFP20) was detected in a segmental duplication block by the PGDD (Plant Genome Duplication Database, http:\\\\/\\\\/chibba.agtec.uga.edu\\\\/duplication\\\\/) as a putative paralogue of At2g30395 (AtOFP17; see below for details).\\n(#3) All search results were collated and manually curated to produce a nonredundant data set that was then subjected to a CDD (Conserved Domain Database) search (http:\\\\/\\\\/www.ncbi.nlm.nih.gov\\\\/ Structure\\\\/cdd\\\\/cdd.shtml; Marchler-Baueret al., 2011) for the verification of the OVATE domain (PF04844).\\n(#4) All sequences were thus verified, with the exception of SlOFP16, as well as AtOFP17, AtOFP20 and their orthologues.\\n(#5) SlOFP16 no longer seems to possess the OVATE domain, whereas the AtOFP17\\\\/ 20-like proteins have a divergent but still recognizable OVATE domain in the multiple alignment with other OVATE proteins (Supplementary Data File S2).\\n(#6) Excluding SlOFP16, a total of 265 putative OVATE protein sequences were obtained from the 13 selected plant genomes.\\n(#7) The nomenclature of all the putative OVATE proteins is listed in Supplementary Data Table S2 .'},\n"," {'id': 36218,\n","  'text': 'SKIP\\n(#1) In order to estimate the evolutionary rate for each sub-group, first whole cDNA sequences from each sub-group were retrieved and aligned with the corresponding alignment of protein sequences as reference using PAL2NAL (Suyama et al., 2006) .\\n(#2) Then, aligned codon sequences encoding the OVATE domain were selected for estimating the ratio of dN (the number of nonsynonymous substitutions per non-synonymous site) to dS (the number of synonymous substitutions per synonymous site), i.e.\\n(#3) the v ratio, by the CODEML implemented in PAML 4.6 (Yang, 2007) .\\n(#4) The average v ratio of each group was estimated with the one-ratio site model M0 (NSsites ¼ 0, model ¼ 0), which assumes that all sites across the phylogeny evolve with the same v ratio.\\n(#5) To determine whether some branches evolve under positive selection (v .\\n(#6) 1), the v ratio of each branch was estimated under the assumption of a free ratios model (NSsites ¼ 0 and model ¼ 1).',\n","  'label': [[0, 4, 'Bad parse'],\n","   [0, 4, 'Bad structure'],\n","   [6, 9, 'Score 4'],\n","   [6, 9, 'Supporting detail sentence'],\n","   [252, 257, 'Score 2'],\n","   [253, 256, 'Supporting detail sentence'],\n","   [500, 502, 'Supporting detail sentence'],\n","   [500, 503, 'Score 1'],\n","   [571, 573, 'Supporting detail sentence'],\n","   [571, 574, 'Score 3'],\n","   [763, 765, 'Score 1'],\n","   [763, 765, 'Supporting detail sentence'],\n","   [840, 842, 'Supporting detail sentence'],\n","   [840, 843, 'Score 1']],\n","  'previous_para': 'SKIP\\n(#1) Multiple sequence alignment of all 265 full-length putative OVATE protein sequences was performed using MUSCLE (http:\\\\/\\\\/www.ebi.ac.uk\\\\/Tools\\\\/msa\\\\/muscle\\\\/; Edgar, 2004) .\\n(#2) PtOFP5 and ZmOFP42 were re-annotated because of an insertion of 59 amino acids in the conserved OVATE domain (see Supplementary Data File S1).\\n(#3) Given the high degree of diversity among the full-length OVATE protein sequences, phylogenetic analysis was conducted on the alignment of the conserved OVATE domain.\\n(#4) Alignments were manually revised in BioEdit (Hall, 1999) , and the resulting alignment of 86 amino acid sites was used for maximum likelihood (ML) analysis using RAxML 7.0.4 (Stamatakis, 2006) , with the BLOSUM62 amino acid substitution matrix and CAT approximation, and 200 bootstrap replicates.\\n(#5) The phylogenetic relationships of OVATE proteins in arabidopsis, rice and tomato were also estimated separately using MEGA 4.0 (Kumar et al., 2008) with the Neighbor -Joining (NJ) method based on the p-distance model, pairwise deletion and 1000 bootstraps.\\n(#6) Trees were visually inspected and edited in Dendroscope (Huson et al., 2007) .\\n(#7) Since there was no prior information on placement of the root, trees were rooted with midpoint rooting.\\n(#8) The MEME (version 4.9.0) motif search tool (http:\\\\/\\\\/meme.nbcr.net\\\\/meme\\\\/ intro.html; Bailey et al., 2009) was used to detect conserved motifs within related OFPs with the following parameters: maximum number of motifs: 10; 4 ≤ motif width ≤ 70; 2 ≤ number of sites ≤ 300.\\n(#9) The sequence logos and E-values for each motif are indicated in Supplementary Data Fig.\\n(#10) S1 .'},\n"," {'id': 36219,\n","  'text': 'SKIP\\n(#1) The ML tree based on the conserved OVATE domain was reconciled with the known species tree in Notung 2.6 (Durand et al., 2006; Vernot et al., 2008) to infer possible gene duplication and loss events.\\n(#2) Rearrangement producing alternative event histories with a minimum duplication\\\\/loss score was performed to avoid overestimation of gene turnover along all lineages.',\n","  'label': [[0, 4, 'Too short']],\n","  'previous_para': 'SKIP\\n(#1) In order to estimate the evolutionary rate for each sub-group, first whole cDNA sequences from each sub-group were retrieved and aligned with the corresponding alignment of protein sequences as reference using PAL2NAL (Suyama et al., 2006) .\\n(#2) Then, aligned codon sequences encoding the OVATE domain were selected for estimating the ratio of dN (the number of nonsynonymous substitutions per non-synonymous site) to dS (the number of synonymous substitutions per synonymous site), i.e.\\n(#3) the v ratio, by the CODEML implemented in PAML 4.6 (Yang, 2007) .\\n(#4) The average v ratio of each group was estimated with the one-ratio site model M0 (NSsites ¼ 0, model ¼ 0), which assumes that all sites across the phylogeny evolve with the same v ratio.\\n(#5) To determine whether some branches evolve under positive selection (v .\\n(#6) 1), the v ratio of each branch was estimated under the assumption of a free ratios model (NSsites ¼ 0 and model ¼ 1).'},\n"," {'id': 36220,\n","  'text': 'SKIP\\n(#1) Chromosomal localization of OVATE genes in arabidopsis, rice and tomato The chromosomal localization of AtOFP genes in arabidopsis was visualized using the Chromosome Map Tool available at TAIR (http:\\\\/\\\\/www.arabidopsis.org\\\\/jsp\\\\/ChromosomeMap\\\\/tool.\\n(#2) jsp).\\n(#3) The chromosomal locations of OsOFP genes in rice and SlOFP genes in tomato were generated by the rice genome browser at the MSU RGAP (Rice Genome Annotation Project) database (http:\\\\/\\\\/rice.plantbiology.msu.edu\\\\/cgi-bin\\\\/gbrowse\\\\/rice\\\\/), and the tomato genome browser at the SGN (Sol Genome Network) database (http:\\\\/\\\\/solgenomics.net\\\\/gbrowse\\\\/bin\\\\/gbrowse\\\\/ ITAG2_genomic\\\\/), respectively.',\n","  'label': [[0, 4, 'Bad parse'], [0, 4, 'Too short']],\n","  'previous_para': 'SKIP\\n(#1) The ML tree based on the conserved OVATE domain was reconciled with the known species tree in Notung 2.6 (Durand et al., 2006; Vernot et al., 2008) to infer possible gene duplication and loss events.\\n(#2) Rearrangement producing alternative event histories with a minimum duplication\\\\/loss score was performed to avoid overestimation of gene turnover along all lineages.'},\n"," {'id': 36221,\n","  'text': 'SKIP\\n(#1) The pattern of OVATE gene duplication resulting in the expansion of the OVATE gene family was determined in arabidopsis, rice and tomato.\\n(#2) If paralogues were either adjacent or separated by ≤ 5 genes along a chromosome they were assigned as duplicates by tandem duplication (Zhao et al., 2010; Xia et al., 2011) .\\n(#3) If paralogues were within known genomic duplication blocks, they were considered to be duplicated through segmental duplication.\\n(#4) The Locus Search Tool at the PGGD (Plant Genome Duplication Database, http:\\\\/\\\\/chibba.agtec.uga.edu\\\\/duplication\\\\/) was used to determine if the AtOFP genes, OsOFP genes and SlOFP genes existed within genomic duplication blocks.\\n(#5) The presence of OsOFP genes in segmental duplication blocks was also ZmOFP9 ZmOFP4.',\n","  'label': [[0, 4, 'Bad structure'],\n","   [6, 9, 'Topic sentence'],\n","   [7, 8, 'Score 3'],\n","   [10, 47, 'Paragraph Topic'],\n","   [149, 152, 'Score 3'],\n","   [149, 152, 'Supporting detail sentence'],\n","   [328, 331, 'Score 3'],\n","   [329, 331, 'Supporting detail sentence'],\n","   [463, 465, 'Supporting detail sentence'],\n","   [463, 466, 'Score 3'],\n","   [692, 695, 'Supporting detail sentence'],\n","   [693, 697, 'Score 2']],\n","  'previous_para': 'SKIP\\n(#1) Chromosomal localization of OVATE genes in arabidopsis, rice and tomato The chromosomal localization of AtOFP genes in arabidopsis was visualized using the Chromosome Map Tool available at TAIR (http:\\\\/\\\\/www.arabidopsis.org\\\\/jsp\\\\/ChromosomeMap\\\\/tool.\\n(#2) jsp).\\n(#3) The chromosomal locations of OsOFP genes in rice and SlOFP genes in tomato were generated by the rice genome browser at the MSU RGAP (Rice Genome Annotation Project) database (http:\\\\/\\\\/rice.plantbiology.msu.edu\\\\/cgi-bin\\\\/gbrowse\\\\/rice\\\\/), and the tomato genome browser at the SGN (Sol Genome Network) database (http:\\\\/\\\\/solgenomics.net\\\\/gbrowse\\\\/bin\\\\/gbrowse\\\\/ ITAG2_genomic\\\\/), respectively.'},\n"," {'id': 36223,\n","  'text': 'SKIP\\n(#1) OsOFP12 investigated using the rice segmental duplication block database at MSU RGAP (http:\\\\/\\\\/rice.plantbiology.msu.edu\\\\/segmental_dup\\\\/ index.shtml).\\n(#2) Given that synonymous substitution rates (Ks) within duplicated genes are postulated to be similar, Ks values can be used as the proxy for time (Shiu et al., 2004) .\\n(#3) The Ks values of paralogues in segmental duplication blocks were retrieved from the PGDD or, in the case of OsOFP23 and OsOFP30, from the rice segmental duplication block database at MSU RGAP computed by the KaKs_calculator (Zhang et al., 2006) .\\n(#4) The timing of duplication events can be estimated using the Ks value and a given clock-like rate, l, through the formula T ¼ Ks\\\\/2l, where for arabidopsis and tomato, l ¼ 1 .\\n(#5) 5 × 10 -8.',\n","  'label': [[0, 4, 'Bad parse'],\n","   [0, 4, 'Bad structure'],\n","   [5, 8, 'Score 1'],\n","   [159, 162, 'Score 3'],\n","   [159, 163, 'Supporting detail sentence'],\n","   [330, 332, 'Score 3'],\n","   [330, 333, 'Supporting detail sentence'],\n","   [582, 584, 'Score 3'],\n","   [582, 585, 'Supporting detail sentence'],\n","   [761, 763, 'Score 1']],\n","  'previous_para': 'SKIP\\n(#1) The pattern of OVATE gene duplication resulting in the expansion of the OVATE gene family was determined in arabidopsis, rice and tomato.\\n(#2) If paralogues were either adjacent or separated by ≤ 5 genes along a chromosome they were assigned as duplicates by tandem duplication (Zhao et al., 2010; Xia et al., 2011) .\\n(#3) If paralogues were within known genomic duplication blocks, they were considered to be duplicated through segmental duplication.\\n(#4) The Locus Search Tool at the PGGD (Plant Genome Duplication Database, http:\\\\/\\\\/chibba.agtec.uga.edu\\\\/duplication\\\\/) was used to determine if the AtOFP genes, OsOFP genes and SlOFP genes existed within genomic duplication blocks.\\n(#5) The presence of OsOFP genes in segmental duplication blocks was also ZmOFP9 ZmOFP4.'},\n"," {'id': 36224,\n","  'text': 'SKIP\\n(#1) substitutions per synonymous site per year, while for rice, l ¼ 6 .\\n(#2) 5 × 10 -9 substitutions per synonymous site per year (Blanc and Wolfe, 2004; Liu et al., 2010) .',\n","  'label': [[0, 4, 'Bad parse'], [0, 4, 'Too short']],\n","  'previous_para': 'SKIP\\n(#1) OsOFP12 investigated using the rice segmental duplication block database at MSU RGAP (http:\\\\/\\\\/rice.plantbiology.msu.edu\\\\/segmental_dup\\\\/ index.shtml).\\n(#2) Given that synonymous substitution rates (Ks) within duplicated genes are postulated to be similar, Ks values can be used as the proxy for time (Shiu et al., 2004) .\\n(#3) The Ks values of paralogues in segmental duplication blocks were retrieved from the PGDD or, in the case of OsOFP23 and OsOFP30, from the rice segmental duplication block database at MSU RGAP computed by the KaKs_calculator (Zhang et al., 2006) .\\n(#4) The timing of duplication events can be estimated using the Ks value and a given clock-like rate, l, through the formula T ¼ Ks\\\\/2l, where for arabidopsis and tomato, l ¼ 1 .\\n(#5) 5 × 10 -8.'},\n"," {'id': 36225,\n","  'text': \"SKIP\\n(#1) Quantitative real-time PCR was performed on a round-fruited tomato variety 'Stupicke'.\\n(#2) Total RNA was isolated from root, leaf, sepal, petal, stamen, pistil, green fruit and red fruit tissue (Trizol, Invitrogen).\\n(#3) First-strand cDNA was synthesized using the PrimeScript w RT reagent Kit with gDNA Eraser (TaKaRa).\\n(#4) Quantitative real-time PCRs were carried out in 20 mL with 2 mL of first-strand cDNA (20 ng mL -1 ), 10 mL of 2 × SYBR Premix w ExTaq TM (TaKaRa), 0 .\\n(#5) 5 mL of 50 × ROX reference dye II (TaKaRa) and 0 .\\n(#6) 8 mL of both forward and reverse primers (10 mM) in an ABI PRISM 7500 system (Applied Biosystems).\\n(#7) Cycling conditions were 95 8C for 10 s followed by 40 cycles of 95 8C for 5 s and 60 8C for 34 s.\\n(#8) Melting curve analysis was used to confirm the specificity of amplification.\\n(#9) Tomato Actin was used as the endogenous control.\\n(#10) The primers used in the assay are listed in Supplementary Data Table  S3 .\\n(#11) Three technical replicates were carried out for each PCR.\\n(#12) Relative expression was calculated using the 2 2DDCT method as previously described (Livak and Schmittgen, 2001 ).\\n(#13) All expression levels were shown as relative to the expression of tomato SlOFP9 in sepals, which exhibited an average expression level.\\n(#14) A heatmap was visualized using Genesis software (Sturn et al., 2002) representing the log2 value of the relative expression level.\\n(#15) Bar charts for the relative expression level of each SlOFP gene are shown in Supplementary Data Fig.\\n(#16) S2 .\\n(#17) There is a possibility that the differences in relative expression between the different genes are due to unequal efficiencies in the amplification of the target sequences.\",\n","  'label': [[0, 4, 'Bad structure'],\n","   [6, 8, 'Score 3'],\n","   [6, 9, 'Supporting detail sentence'],\n","   [98, 100, 'Score 3'],\n","   [98, 100, 'Supporting detail sentence'],\n","   [228, 230, 'Supporting detail sentence'],\n","   [228, 231, 'Score 3'],\n","   [333, 335, 'Score 3'],\n","   [333, 335, 'Supporting detail sentence'],\n","   [545, 547, 'Supporting detail sentence'],\n","   [545, 548, 'Score 1'],\n","   [649, 651, 'Score 3'],\n","   [649, 651, 'Supporting detail sentence'],\n","   [752, 754, 'Score 3'],\n","   [752, 754, 'Supporting detail sentence'],\n","   [833, 836, 'Score 3'],\n","   [834, 836, 'Supporting detail sentence'],\n","   [888, 891, 'Score 3'],\n","   [888, 891, 'Supporting detail sentence'],\n","   [969, 972, 'Score 3'],\n","   [969, 972, 'Supporting detail sentence'],\n","   [1033, 1036, 'Score 3'],\n","   [1033, 1036, 'Supporting detail sentence'],\n","   [1154, 1157, 'Supporting detail sentence'],\n","   [1155, 1157, 'Score 3'],\n","   [1296, 1299, 'Score 3'],\n","   [1296, 1299, 'Supporting detail sentence'],\n","   [1433, 1436, 'Score 3'],\n","   [1433, 1436, 'Supporting detail sentence'],\n","   [1551, 1554, 'Score 3'],\n","   [1551, 1554, 'Supporting detail sentence']],\n","  'previous_para': 'SKIP\\n(#1) substitutions per synonymous site per year, while for rice, l ¼ 6 .\\n(#2) 5 × 10 -9 substitutions per synonymous site per year (Blanc and Wolfe, 2004; Liu et al., 2010) .'},\n"," {'id': 36226,\n","  'text': 'SKIP\\n(#1) OVATE-like proteins were identified in all 13 plant genomes selected to represent the major evolutionary lineages of land plants including the early-diverged land plants Physcomitrella patens (moss) and Selaginella moellendorffii (spikemoss) (Fig.\\n(#2) 1) .\\n(#3) In general, monocots had more OFPs than eudicots, with Zea mays (maize) containing the largest number (i.e.\\n(#4) 45) of OVATE proteins.\\n(#5) With respect to the core eudicots, thegenomes of Mimulus guttatus (monkey flower), Solanum lycopersicum (tomato) and Solanum tuberosum ( potato) from the asterid clade contained similar numbers of OVATE proteins.\\n(#6) In the rosid clade, the OVATE protein numbers in different species vary from nine in Vitis vinifera (grape vine) to 29 in Populus trichocarpa ( poplar) (Fig.\\n(#7) 1) .\\n(#8) It is worth noting that a previously unnamed AtOFP, At2g36026 (designated AtOFP19), and an uncharacterized AtOFP member, At1g06923 (designated AtOFP20), were included in this study.\\n(#9) AtOFP19 possesses the conserved OVATE domain and is annotated as an OVATE family protein in TAIR (http:\\\\/\\\\/www.arabidopsis.org\\\\/).\\n(#10) AtOFP20 is annotated as being most closely related to AtOFP17 (At2g30395), consistent with both genes being putative paralogues occurring within segmental duplication blocks detected using the PGDD.\\n(#11) Although AtOFP17 is annotated as an OVATE family protein, the presence of a conserved OVATE domain was not detected in a CDD search.\\n(#12) The absence of the OVATE domain is prevalent in both its paralogue AtOFP20 and their putative orthologues in other species, which cluster with AtOFP17 into a distinct group (sub-group 8; Fig.\\n(#13) 2 ).\\n(#14) Here, AtOFP17-like members are included as OVATE proteins on the basis that the OVATE domain in the AtOFP17 sub-group seems to be quite obvious from the alignment that was used for the phylogenetic analysis (Supplementary Data File S2), as well as the fact that AtOFP17 was annotated as an OVATE-containing protein despite relatively low sequence similarity with other AtOFPs determined by Wang et al.\\n(#15) (2011) .',\n","  'label': [[0, 4, 'Bad parse'],\n","   [0, 4, 'Bad structure'],\n","   [6, 8, 'Score 3'],\n","   [6, 8, 'Topic sentence'],\n","   [10, 29, 'Paragraph Topic'],\n","   [259, 261, 'Score 1'],\n","   [269, 271, 'Score 1'],\n","   [269, 271, 'Supporting detail sentence'],\n","   [382, 384, 'Score 1'],\n","   [382, 384, 'Score 1'],\n","   [393, 407, 'Paragraph Topic'],\n","   [410, 412, 'Score 3'],\n","   [410, 412, 'Supporting detail sentence'],\n","   [611, 625, 'Paragraph Topic'],\n","   [628, 630, 'Supporting detail sentence'],\n","   [628, 631, 'Score 1'],\n","   [655, 669, 'Paragraph Topic'],\n","   [791, 793, 'Score 1'],\n","   [801, 803, 'Score 3'],\n","   [801, 803, 'Supporting detail sentence'],\n","   [850, 855, 'Topic attribute 1'],\n","   [879, 884, 'Topic attribute 1'],\n","   [912, 917, 'Topic attribute 1'],\n","   [948, 953, 'Topic attribute 1'],\n","   [988, 990, 'Score 3'],\n","   [988, 990, 'Supporting detail sentence'],\n","   [992, 999, 'Topic attribute 1'],\n","   [1024, 1029, 'Paragraph Topic'],\n","   [1121, 1124, 'Score 3'],\n","   [1121, 1124, 'Supporting detail sentence'],\n","   [1126, 1131, 'Topic attribute 1'],\n","   [1180, 1185, 'Topic attribute 1'],\n","   [1326, 1329, 'Score 3'],\n","   [1326, 1329, 'Supporting detail sentence'],\n","   [1340, 1347, 'Topic attribute 1'],\n","   [1367, 1372, 'Paragraph Topic'],\n","   [1465, 1468, 'Score 3'],\n","   [1465, 1468, 'Supporting detail sentence'],\n","   [1489, 1501, 'Paragraph Topic'],\n","   [1537, 1542, 'Topic attribute 1'],\n","   [1613, 1618, 'Topic attribute 1'],\n","   [1663, 1666, 'Score 1'],\n","   [1674, 1677, 'Score 2'],\n","   [1674, 1677, 'Supporting detail sentence'],\n","   [1685, 1692, 'Topic attribute 1'],\n","   [1722, 1727, 'Paragraph Topic'],\n","   [1759, 1764, 'Paragraph Topic'],\n","   [1780, 1784, 'Topic attribute 1'],\n","   [1941, 1948, 'Topic attribute 1'],\n","   [1969, 1974, 'Paragraph Topic'],\n","   [2048, 2053, 'Topic attribute 1'],\n","   [2082, 2085, 'Score 1']],\n","  'previous_para': \"SKIP\\n(#1) Quantitative real-time PCR was performed on a round-fruited tomato variety 'Stupicke'.\\n(#2) Total RNA was isolated from root, leaf, sepal, petal, stamen, pistil, green fruit and red fruit tissue (Trizol, Invitrogen).\\n(#3) First-strand cDNA was synthesized using the PrimeScript w RT reagent Kit with gDNA Eraser (TaKaRa).\\n(#4) Quantitative real-time PCRs were carried out in 20 mL with 2 mL of first-strand cDNA (20 ng mL -1 ), 10 mL of 2 × SYBR Premix w ExTaq TM (TaKaRa), 0 .\\n(#5) 5 mL of 50 × ROX reference dye II (TaKaRa) and 0 .\\n(#6) 8 mL of both forward and reverse primers (10 mM) in an ABI PRISM 7500 system (Applied Biosystems).\\n(#7) Cycling conditions were 95 8C for 10 s followed by 40 cycles of 95 8C for 5 s and 60 8C for 34 s.\\n(#8) Melting curve analysis was used to confirm the specificity of amplification.\\n(#9) Tomato Actin was used as the endogenous control.\\n(#10) The primers used in the assay are listed in Supplementary Data Table  S3 .\\n(#11) Three technical replicates were carried out for each PCR.\\n(#12) Relative expression was calculated using the 2 2DDCT method as previously described (Livak and Schmittgen, 2001 ).\\n(#13) All expression levels were shown as relative to the expression of tomato SlOFP9 in sepals, which exhibited an average expression level.\\n(#14) A heatmap was visualized using Genesis software (Sturn et al., 2002) representing the log2 value of the relative expression level.\\n(#15) Bar charts for the relative expression level of each SlOFP gene are shown in Supplementary Data Fig.\\n(#16) S2 .\\n(#17) There is a possibility that the differences in relative expression between the different genes are due to unequal efficiencies in the amplification of the target sequences.\"},\n"," {'id': 36227,\n","  'text': 'SKIP\\n(#1) AtOFP9 has been re-annotated to include an additional open reading frame (ORF) encoding a new protein of 129 amino acids (NP_191312) that is distinct from the 411 amino acid protein in the study of Wang et al.\\n(#2) (2011) .\\n(#3) Moreover, this newly annotated AtOFP9 protein has lost the conserved OVATE domain that characterizes the OFPs.\\n(#4) Given this, the newly annotated AtOFP9 should no longer be considered as an OVATE protein in arabidopsis, and we propose that there are 19 arabidopsis OVATE proteins instead of the 18 originally described (Wang et al., 2007 .',\n","  'label': [[6, 8, 'Score 4'],\n","   [6, 8, 'Topic sentence'],\n","   [10, 16, 'Paragraph Topic'],\n","   [235, 237, 'Score 4'],\n","   [235, 237, 'Supporting detail sentence'],\n","   [270, 276, 'Paragraph Topic'],\n","   [308, 313, 'Topic attribute 1'],\n","   [344, 348, 'Topic attribute 2'],\n","   [351, 353, 'Score 4'],\n","   [351, 354, 'Conclusion sentence'],\n","   [387, 393, 'Paragraph Topic'],\n","   [431, 436, 'Topic attribute 1'],\n","   [506, 511, 'Topic attribute 1'],\n","   [0, 4, 'Bad parse']],\n","  'previous_para': 'SKIP\\n(#1) OVATE-like proteins were identified in all 13 plant genomes selected to represent the major evolutionary lineages of land plants including the early-diverged land plants Physcomitrella patens (moss) and Selaginella moellendorffii (spikemoss) (Fig.\\n(#2) 1) .\\n(#3) In general, monocots had more OFPs than eudicots, with Zea mays (maize) containing the largest number (i.e.\\n(#4) 45) of OVATE proteins.\\n(#5) With respect to the core eudicots, thegenomes of Mimulus guttatus (monkey flower), Solanum lycopersicum (tomato) and Solanum tuberosum ( potato) from the asterid clade contained similar numbers of OVATE proteins.\\n(#6) In the rosid clade, the OVATE protein numbers in different species vary from nine in Vitis vinifera (grape vine) to 29 in Populus trichocarpa ( poplar) (Fig.\\n(#7) 1) .\\n(#8) It is worth noting that a previously unnamed AtOFP, At2g36026 (designated AtOFP19), and an uncharacterized AtOFP member, At1g06923 (designated AtOFP20), were included in this study.\\n(#9) AtOFP19 possesses the conserved OVATE domain and is annotated as an OVATE family protein in TAIR (http:\\\\/\\\\/www.arabidopsis.org\\\\/).\\n(#10) AtOFP20 is annotated as being most closely related to AtOFP17 (At2g30395), consistent with both genes being putative paralogues occurring within segmental duplication blocks detected using the PGDD.\\n(#11) Although AtOFP17 is annotated as an OVATE family protein, the presence of a conserved OVATE domain was not detected in a CDD search.\\n(#12) The absence of the OVATE domain is prevalent in both its paralogue AtOFP20 and their putative orthologues in other species, which cluster with AtOFP17 into a distinct group (sub-group 8; Fig.\\n(#13) 2 ).\\n(#14) Here, AtOFP17-like members are included as OVATE proteins on the basis that the OVATE domain in the AtOFP17 sub-group seems to be quite obvious from the alignment that was used for the phylogenetic analysis (Supplementary Data File S2), as well as the fact that AtOFP17 was annotated as an OVATE-containing protein despite relatively low sequence similarity with other AtOFPs determined by Wang et al.\\n(#15) (2011) .'},\n"," {'id': 36228,\n","  'text': 'SKIP\\n(#1) To elucidate further the evolutionary history and origin of the OVATE protein family in green plants (Viridiplantae), we also screened the genomes of the chlorophytes, Volvox carteri and Chlamydomonas reinhardtii, for OVATE proteins.\\n(#2) No proteins with similarity to OFPs were found in the two chlorophyte genomes, which represent the ancestral algal relatives of the Viridiplantae.',\n","  'label': [[0, 4, 'Too short']],\n","  'previous_para': 'SKIP\\n(#1) AtOFP9 has been re-annotated to include an additional open reading frame (ORF) encoding a new protein of 129 amino acids (NP_191312) that is distinct from the 411 amino acid protein in the study of Wang et al.\\n(#2) (2011) .\\n(#3) Moreover, this newly annotated AtOFP9 protein has lost the conserved OVATE domain that characterizes the OFPs.\\n(#4) Given this, the newly annotated AtOFP9 should no longer be considered as an OVATE protein in arabidopsis, and we propose that there are 19 arabidopsis OVATE proteins instead of the 18 originally described (Wang et al., 2007 .'},\n"," {'id': 36229,\n","  'text': \"SKIP\\n(#1) To obtain an overview of the phylogenetic relationships between plant OFPs, phylogenetic trees were reconstructed based on the alignment of the conserved OVATE regions (Fig.\\n(#2) 2) .\\n(#3) OVATE proteins from different flowering plant (angiosperms) species cluster together, while bryophyte and lycophyte OVATE proteins form distinct, predominantly lineage-specific clades (Fig.\\n(#4) 2) .\\n(#5) Eleven sub-groups of angiosperm OVATE proteins were defined according to the topology of the phylogenetic trees (Fig.\\n(#6) 2) .\\n(#7) These sub-groups were also supported by the presence of conserved motifs that are outside the OVATE domain and unique to each sub-group detected by the MEME program ( Fig.\\n(#8) 3; Supplementary Data Fig.\\n(#9) S1 ).\\n(#10) Although some of the 11 sub-groups have low bootstrap support due to the fact that only 86 amino acid sites in the OVATE domain were used for inferring the phylogeny, the presence of conserved signature motifs unique to each sub-group corroborates our phylogenetic analysis ( Fig.\\n(#11) 3 ; Supplementary Data Fig.\\n(#12) S1 ).\\n(#13) The OVATE proteins from mosses and lycophytes were not classified further, as they do not form well-defined sub-groups with the angiosperm OVATE proteins.\\n(#14) Angiosperm OVATE proteins from different species grouped into compact clades.\\n(#15) This possibly resulted from rapid expansion of OVATE proteins in early seed plants or angiosperms following their divergence from the early-diverged land plant species.\\n(#16) Many gene families within angiosperms have experienced rapid expansion during evolution and adaption to various environments (Corrêa et al., 2008) .\\n(#17) Both monocot and eudicot OVATE proteins expanded rapidly within sub-groups 1, 4 and 5 to form a lineage-specific clade (e.g.\\n(#18) a Solanum-specific clade containing exclusively OVATE proteins from S.\\n(#19) lycopersicum and S.\\n(#20) tuberosum; Fig.\\n(#21) 2 ).\\n(#22) Of the 11 angiosperm OVATE sub-groups, sub-group 10 was specific to monocots, while sub-group 11 was an 'orphan' group specific to eudicots and included only three members, AcOFP10, PpaOFP15 and PtOFP24 (Fig.\\n(#23) 2) .\",\n","  'label': [[0, 4, 'Bad parse'],\n","   [0, 4, 'Bad structure'],\n","   [6, 8, 'Score 3'],\n","   [6, 8, 'Supporting detail sentence'],\n","   [80, 84, 'Paragraph Topic'],\n","   [185, 187, 'Score 1'],\n","   [195, 197, 'Score 3'],\n","   [195, 197, 'Supporting detail sentence'],\n","   [199, 213, 'Paragraph Topic'],\n","   [391, 392, 'Score 1'],\n","   [400, 402, 'Score 3'],\n","   [400, 402, 'Supporting detail sentence'],\n","   [436, 451, 'Paragraph Topic'],\n","   [523, 525, 'Score 1'],\n","   [532, 535, 'Supporting detail sentence'],\n","   [533, 535, 'Score 3'],\n","   [631, 636, 'Paragraph Topic'],\n","   [710, 712, 'Score 1'],\n","   [742, 744, 'Score 1'],\n","   [753, 756, 'Score 3'],\n","   [753, 756, 'Supporting detail sentence'],\n","   [873, 885, 'Paragraph Topic'],\n","   [1040, 1043, 'Score 1'],\n","   [1074, 1077, 'Score 1'],\n","   [1086, 1089, 'Score 3'],\n","   [1086, 1089, 'Supporting detail sentence'],\n","   [1095, 1109, 'Paragraph Topic'],\n","   [1230, 1244, 'Paragraph Topic'],\n","   [1247, 1250, 'Supporting detail sentence'],\n","   [1248, 1250, 'Score 3'],\n","   [1263, 1277, 'Paragraph Topic'],\n","   [1331, 1334, 'Score 3'],\n","   [1331, 1334, 'Supporting detail sentence'],\n","   [1383, 1397, 'Paragraph Topic'],\n","   [1506, 1509, 'Score 3'],\n","   [1506, 1509, 'Supporting detail sentence'],\n","   [1661, 1664, 'Score 3'],\n","   [1661, 1664, 'Supporting detail sentence'],\n","   [1691, 1706, 'Paragraph Topic'],\n","   [1792, 1795, 'Score 1'],\n","   [1845, 1859, 'Paragraph Topic'],\n","   [1869, 1872, 'Score 1'],\n","   [1896, 1898, 'Score 1'],\n","   [1917, 1920, 'Score 1'],\n","   [1928, 1931, 'Supporting detail sentence'],\n","   [1928, 1932, 'Score 3'],\n","   [1954, 1959, 'Paragraph Topic'],\n","   [2143, 2146, 'Score 1']],\n","  'previous_para': 'SKIP\\n(#1) To elucidate further the evolutionary history and origin of the OVATE protein family in green plants (Viridiplantae), we also screened the genomes of the chlorophytes, Volvox carteri and Chlamydomonas reinhardtii, for OVATE proteins.\\n(#2) No proteins with similarity to OFPs were found in the two chlorophyte genomes, which represent the ancestral algal relatives of the Viridiplantae.'},\n"," {'id': 36230,\n","  'text': 'SKIP\\n(#1) In order to clarify the evolution of the OVATE family further, we examined the distribution of OFPs from different plant species within each sub-group (Table 1) .\\n(#2) We suggest two different mechanisms to account for the observed evolutionary pattern, namely conserved evolution and divergent expansion, similar to what was proposed for the evolution of F-box genes in plants (Xu et al., 2009) .\\n(#3) Within sub-group 8, sequences are conserved in all 45  33  11  16  29  19  11  9  23  26  25  Sub-group1  9  7  1  2  5  3  2  1  4  9  9  Sub-group2  2  2  2  2  4  3  2  1  2  2  1  Sub-group3  3  1  1  1  1  1  1  1  3  2  1  Sub-group4  6  5  1  3  6  2  3  3  4  5  5  Sub-group5  12  8  1  2  4  2  --4  3  3  Sub-group6  3  2  1  2  4  4  1  1  2  3  3  Sub-group7  2  2  1  1  1  1  1  1  2  1  1  Sub-group8  2  2  1  1  2  2  1  1  1  1  2  Sub-group9  2  1  1  1 .',\n","  'label': [[0, 4, 'Bad structure'],\n","   [6, 9, 'Score 3'],\n","   [6, 9, 'Supporting detail sentence'],\n","   [51, 63, 'Paragraph Topic'],\n","   [105, 109, 'Paragraph Topic'],\n","   [174, 177, 'Score 3'],\n","   [174, 177, 'Supporting detail sentence'],\n","   [409, 411, 'Score 1'],\n","   [409, 411, 'Supporting detail sentence'],\n","   [0, 4, 'Bad parse']],\n","  'previous_para': \"SKIP\\n(#1) To obtain an overview of the phylogenetic relationships between plant OFPs, phylogenetic trees were reconstructed based on the alignment of the conserved OVATE regions (Fig.\\n(#2) 2) .\\n(#3) OVATE proteins from different flowering plant (angiosperms) species cluster together, while bryophyte and lycophyte OVATE proteins form distinct, predominantly lineage-specific clades (Fig.\\n(#4) 2) .\\n(#5) Eleven sub-groups of angiosperm OVATE proteins were defined according to the topology of the phylogenetic trees (Fig.\\n(#6) 2) .\\n(#7) These sub-groups were also supported by the presence of conserved motifs that are outside the OVATE domain and unique to each sub-group detected by the MEME program ( Fig.\\n(#8) 3; Supplementary Data Fig.\\n(#9) S1 ).\\n(#10) Although some of the 11 sub-groups have low bootstrap support due to the fact that only 86 amino acid sites in the OVATE domain were used for inferring the phylogeny, the presence of conserved signature motifs unique to each sub-group corroborates our phylogenetic analysis ( Fig.\\n(#11) 3 ; Supplementary Data Fig.\\n(#12) S1 ).\\n(#13) The OVATE proteins from mosses and lycophytes were not classified further, as they do not form well-defined sub-groups with the angiosperm OVATE proteins.\\n(#14) Angiosperm OVATE proteins from different species grouped into compact clades.\\n(#15) This possibly resulted from rapid expansion of OVATE proteins in early seed plants or angiosperms following their divergence from the early-diverged land plant species.\\n(#16) Many gene families within angiosperms have experienced rapid expansion during evolution and adaption to various environments (Corrêa et al., 2008) .\\n(#17) Both monocot and eudicot OVATE proteins expanded rapidly within sub-groups 1, 4 and 5 to form a lineage-specific clade (e.g.\\n(#18) a Solanum-specific clade containing exclusively OVATE proteins from S.\\n(#19) lycopersicum and S.\\n(#20) tuberosum; Fig.\\n(#21) 2 ).\\n(#22) Of the 11 angiosperm OVATE sub-groups, sub-group 10 was specific to monocots, while sub-group 11 was an 'orphan' group specific to eudicots and included only three members, AcOFP10, PpaOFP15 and PtOFP24 (Fig.\\n(#23) 2) .\"},\n"," {'id': 36231,\n","  'text': 'SKIP\\n(#1) Pp, Physcomitrella patens; Sm, Selaginella moellendorffii; Zm, Zea mays; Os, Oryza sativa; Ac, Aquilegia coerulea; Ppa, Prunus persica; Pt, Populus trichocarpa; At, Arabidopsis thaliana; Cp, Carica papaya; Vv, Vitis vinifera; Mg, Mimulus guttatus; St, Solanum tuberosum; Sl, Solanum lycopersicum.\\n(#2) angiosperm species, with only one or two members from each species.\\n(#3) Sub-groups 3, 7 and 9 also resemble this, comprising primarily one or two OVATE proteins from each plant species, with phylogenetic relationships partially mirroring the plant phylogeny ( Fig.\\n(#4) 2 ; Table 1 ).\\n(#5) As such, the OVATE members of these sub-groups appear to have remained relatively conserved without large-scale gene expansions during plant divergence.\\n(#6) This suggests that they may have crucial functions that are vital to plant development.\\n(#7) For example, AtOFP5, a previously characterized protein interacting with the two TALE homeodomain proteins BLH1 and KNAT3 (Hackbusch et al., 2005) , was reported to act as a regulator of the BELL -KNOX TALE complex required for normal embryo sac development in arabidopsis (Pagnussat et al., 2007) .\\n(#8) In this study, phylogenetic analysis placed AtOFP5 in the highly conserved sub-group 7, consistent with a conserved role in the essential process of female gametophyte development.\\n(#9) Divergent expansion of protein sub-families occurs when members are not under such tight functional constraints and thus may diverge more rapidly between taxa, often adopting species-specific functions (Hamilton et al., 2006) .\\n(#10) Such subfamilies appear to have evolved for more OVATE proteins, with each containing 3 -12 members from some plant species examined and in some cases forming lineage-specific sub-clades ( Fig.\\n(#11) 2; Table 1 ).\\n(#12) AtOFP1 and AtOFP4, the two bestcharacterized OVATE proteins (Wang et al., 2007 Li et al., 2011) , were assigned to sub-group 6 (Fig.\\n(#13) 2) .\\n(#14) This sub-group has four AtOFPs, namely AtOFP1, AtOFP2, AtOFP3 and AtOFP4.\\n(#15) AtOFP1 is suggested to function as an active transcriptional repressor of AtGA20ox1 expression in the GA biosynthesis pathway, suppressing cell elongation (Wang et al., 2007) .\\n(#16) AtOFP4 is also a transcriptional repressor and has been proposed to form a functional complex with KNAT7, an arabidopsis KNOX homeodomain protein, to regulate secondary cell wall formation (Li et al., 2011) .\\n(#17) AtOFP1, AtOFP2 and AtOFP4 were shown to have similar expression patterns, with all expressed in the roots, inflorescent stem, flower buds and young siliques .\\n(#18) Promoter -GUS (b-glucuronidase) fusions for AtOFP1 and AtOFP4 revealed strong GUS activity in the root vascular cylinder and inflorescent stem (Li et al., 2011) .\\n(#19) Furthermore, arabidopsis plants overexpressing AtOFP4 had similar pleiotropic phenotypes to those overexpressing AtOFP1, including dwarfism, ovate-shaped organs and reduced fertility (Li et al., 2011) .\\n(#20) Nevertheless, it was suggested that AtOFP4 may play a more specific role in the differentiation of xylary fibres and interfascicular fibres based on the irx phenotype (Brown et al., 2005) and the thicker interfascicular fibre cell walls seen in the ofp4-2 loss-of-function mutant but not in the ofp1-1 loss-of-function mutant (Li et al., 2011) .\\n(#21) The irx phenotype, described as irregular xylem (irx), is characterized by a collapse of xylem vessels causing defects in the secondary wall and might be indicative of any secondary cell wall mutation (Brown et al., 2005) .\\n(#22) Thus, functional divergence of AtOFP1 and AtOFP4 from a common ancestral gene may have occurred during arabidopsis genome evolution.',\n","  'label': [[0, 4, 'Bad parse'],\n","   [0, 4, 'Bad structure'],\n","   [381, 384, 'Score 3'],\n","   [381, 384, 'Supporting detail sentence'],\n","   [459, 473, 'Paragraph Topic'],\n","   [599, 601, 'Score 3'],\n","   [599, 601, 'Supporting detail sentence'],\n","   [616, 629, 'Paragraph Topic'],\n","   [757, 759, 'Score 3'],\n","   [757, 759, 'Supporting detail sentence'],\n","   [850, 852, 'Supporting detail sentence'],\n","   [850, 853, 'Score 3'],\n","   [867, 873, 'Topic attribute 1'],\n","   [1155, 1157, 'Score 3'],\n","   [1155, 1157, 'Supporting detail sentence'],\n","   [1203, 1209, 'Topic attribute 1'],\n","   [1341, 1343, 'Score 3'],\n","   [1341, 1343, 'Supporting detail sentence'],\n","   [1574, 1577, 'Score 3'],\n","   [1574, 1577, 'Supporting detail sentence'],\n","   [1628, 1642, 'Paragraph Topic'],\n","   [1794, 1797, 'Score 3'],\n","   [1794, 1797, 'Supporting detail sentence'],\n","   [1799, 1805, 'Topic attribute 1'],\n","   [1811, 1816, 'Topic attribute 1'],\n","   [1844, 1858, 'Paragraph Topic'],\n","   [1944, 1947, 'Supporting detail sentence'],\n","   [1944, 1948, 'Score 3'],\n","   [1973, 1979, 'Topic attribute 1'],\n","   [1988, 2021, 'Topic attribute 1'],\n","   [2024, 2027, 'Score 3'],\n","   [2024, 2027, 'Supporting detail sentence'],\n","   [2029, 2035, 'Topic attribute 1'],\n","   [2207, 2210, 'Score 3'],\n","   [2207, 2210, 'Supporting detail sentence'],\n","   [2422, 2425, 'Score 3'],\n","   [2422, 2425, 'Supporting detail sentence'],\n","   [2427, 2452, 'Topic attribute 1'],\n","   [2587, 2590, 'Score 3'],\n","   [2587, 2590, 'Supporting detail sentence'],\n","   [2636, 2653, 'Topic attribute 1'],\n","   [2756, 2759, 'Score 3'],\n","   [2756, 2759, 'Supporting detail sentence'],\n","   [2874, 2880, 'Topic attribute 1'],\n","   [2965, 2968, 'Supporting detail sentence'],\n","   [3006, 3013, 'Topic attribute 1'],\n","   [3317, 3320, 'Score 3'],\n","   [3317, 3320, 'Supporting detail sentence'],\n","   [3547, 3550, 'Conclusion sentence'],\n","   [3547, 3551, 'Score 3'],\n","   [3583, 3600, 'Topic attribute 1']],\n","  'previous_para': 'SKIP\\n(#1) In order to clarify the evolution of the OVATE family further, we examined the distribution of OFPs from different plant species within each sub-group (Table 1) .\\n(#2) We suggest two different mechanisms to account for the observed evolutionary pattern, namely conserved evolution and divergent expansion, similar to what was proposed for the evolution of F-box genes in plants (Xu et al., 2009) .\\n(#3) Within sub-group 8, sequences are conserved in all 45  33  11  16  29  19  11  9  23  26  25  Sub-group1  9  7  1  2  5  3  2  1  4  9  9  Sub-group2  2  2  2  2  4  3  2  1  2  2  1  Sub-group3  3  1  1  1  1  1  1  1  3  2  1  Sub-group4  6  5  1  3  6  2  3  3  4  5  5  Sub-group5  12  8  1  2  4  2  --4  3  3  Sub-group6  3  2  1  2  4  4  1  1  2  3  3  Sub-group7  2  2  1  1  1  1  1  1  2  1  1  Sub-group8  2  2  1  1  2  2  1  1  1  1  2  Sub-group9  2  1  1  1 .'},\n"," {'id': 36237,\n","  'text': 'SKIP\\n(#1) The rice genome has 33 OVATE-encoding genes designated OsOFP genes.\\n(#2) These OsOFP genes are distributed unevenly across the 12 chromosomes of rice, with the highest density on chromosome 1 (eight genes), followed by chromosome 5 with seven OsOFP genes, while chromosomes 6 and 9 have no OFP genes (Fig.\\n(#3) 4A) .\\n(#4) The phylogenetic relationships of OsOFPs within rice conformed to the analysis based on the conserved OVATE region in all 13 plant genomes (Fig.\\n(#5) 5A) .\\n(#6) The OsOFP family could be divided into ten sub-groups, of which sub-group 10 was a monocot-specific clade, and sub-group 5 contained eight members accounting for a quarter of the OsOFP family in rice (Fig.\\n(#7) 5A) .\\n(#8) The genic structures of all OsOFP-coding genes were also examined.\\n(#9) Nearly all OsOFP genes are intronless, except OsOFP10, which contains a small intron (Fig.\\n(#10) 5D ).',\n","  'label': [[0, 4, 'Bad parse'],\n","   [0, 4, 'Bad structure'],\n","   [6, 8, 'Score 3'],\n","   [6, 8, 'Topic sentence'],\n","   [65, 76, 'Paragraph Topic'],\n","   [79, 81, 'Score 3'],\n","   [79, 81, 'Supporting detail sentence'],\n","   [89, 100, 'Paragraph Topic'],\n","   [140, 151, 'Topic attribute 1'],\n","   [189, 200, 'Topic attribute 1'],\n","   [229, 239, 'Topic attribute 1'],\n","   [253, 264, 'Paragraph Topic'],\n","   [272, 283, 'Topic attribute 1'],\n","   [300, 309, 'Topic attribute 2'],\n","   [317, 319, 'Score 1'],\n","   [328, 330, 'Score 3'],\n","   [328, 330, 'Supporting detail sentence'],\n","   [366, 372, 'Paragraph Topic'],\n","   [434, 446, 'Topic attribute 1'],\n","   [478, 480, 'Score 1'],\n","   [489, 491, 'Score 3'],\n","   [489, 491, 'Supporting detail sentence'],\n","   [497, 502, 'Paragraph Topic'],\n","   [672, 677, 'Paragraph Topic'],\n","   [700, 702, 'Score 1'],\n","   [711, 713, 'Score 3'],\n","   [711, 713, 'Supporting detail sentence'],\n","   [743, 748, 'Paragraph Topic'],\n","   [783, 785, 'Score 3'],\n","   [783, 785, 'Supporting detail sentence'],\n","   [798, 803, 'Paragraph Topic'],\n","   [879, 882, 'Score 1']],\n","  'previous_para': 'SKIP\\n(#1) Pp, Physcomitrella patens; Sm, Selaginella moellendorffii; Zm, Zea mays; Os, Oryza sativa; Ac, Aquilegia coerulea; Ppa, Prunus persica; Pt, Populus trichocarpa; At, Arabidopsis thaliana; Cp, Carica papaya; Vv, Vitis vinifera; Mg, Mimulus guttatus; St, Solanum tuberosum; Sl, Solanum lycopersicum.\\n(#2) angiosperm species, with only one or two members from each species.\\n(#3) Sub-groups 3, 7 and 9 also resemble this, comprising primarily one or two OVATE proteins from each plant species, with phylogenetic relationships partially mirroring the plant phylogeny ( Fig.\\n(#4) 2 ; Table 1 ).\\n(#5) As such, the OVATE members of these sub-groups appear to have remained relatively conserved without large-scale gene expansions during plant divergence.\\n(#6) This suggests that they may have crucial functions that are vital to plant development.\\n(#7) For example, AtOFP5, a previously characterized protein interacting with the two TALE homeodomain proteins BLH1 and KNAT3 (Hackbusch et al., 2005) , was reported to act as a regulator of the BELL -KNOX TALE complex required for normal embryo sac development in arabidopsis (Pagnussat et al., 2007) .\\n(#8) In this study, phylogenetic analysis placed AtOFP5 in the highly conserved sub-group 7, consistent with a conserved role in the essential process of female gametophyte development.\\n(#9) Divergent expansion of protein sub-families occurs when members are not under such tight functional constraints and thus may diverge more rapidly between taxa, often adopting species-specific functions (Hamilton et al., 2006) .\\n(#10) Such subfamilies appear to have evolved for more OVATE proteins, with each containing 3 -12 members from some plant species examined and in some cases forming lineage-specific sub-clades ( Fig.\\n(#11) 2; Table 1 ).\\n(#12) AtOFP1 and AtOFP4, the two bestcharacterized OVATE proteins (Wang et al., 2007 Li et al., 2011) , were assigned to sub-group 6 (Fig.\\n(#13) 2) .\\n(#14) This sub-group has four AtOFPs, namely AtOFP1, AtOFP2, AtOFP3 and AtOFP4.\\n(#15) AtOFP1 is suggested to function as an active transcriptional repressor of AtGA20ox1 expression in the GA biosynthesis pathway, suppressing cell elongation (Wang et al., 2007) .\\n(#16) AtOFP4 is also a transcriptional repressor and has been proposed to form a functional complex with KNAT7, an arabidopsis KNOX homeodomain protein, to regulate secondary cell wall formation (Li et al., 2011) .\\n(#17) AtOFP1, AtOFP2 and AtOFP4 were shown to have similar expression patterns, with all expressed in the roots, inflorescent stem, flower buds and young siliques .\\n(#18) Promoter -GUS (b-glucuronidase) fusions for AtOFP1 and AtOFP4 revealed strong GUS activity in the root vascular cylinder and inflorescent stem (Li et al., 2011) .\\n(#19) Furthermore, arabidopsis plants overexpressing AtOFP4 had similar pleiotropic phenotypes to those overexpressing AtOFP1, including dwarfism, ovate-shaped organs and reduced fertility (Li et al., 2011) .\\n(#20) Nevertheless, it was suggested that AtOFP4 may play a more specific role in the differentiation of xylary fibres and interfascicular fibres based on the irx phenotype (Brown et al., 2005) and the thicker interfascicular fibre cell walls seen in the ofp4-2 loss-of-function mutant but not in the ofp1-1 loss-of-function mutant (Li et al., 2011) .\\n(#21) The irx phenotype, described as irregular xylem (irx), is characterized by a collapse of xylem vessels causing defects in the secondary wall and might be indicative of any secondary cell wall mutation (Brown et al., 2005) .\\n(#22) Thus, functional divergence of AtOFP1 and AtOFP4 from a common ancestral gene may have occurred during arabidopsis genome evolution.'},\n"," {'id': 36239,\n","  'text': 'SKIP\\n(#1) It is of interest that overexpression of AtOFP genes with close phylogenetic relationships was previously shown to produce similar phenotypes.\\n(#2) Plants overexpressing AtOFP1, AtOFP2, AtOFP4, AtOFP5 and AtOFP7 resulted in kidney-shaped cotyledons, and round and curled leaves (class I); overexpression of AtOFP6 and AtOFP8 caused flat, thick and cyan leaves (class II); and overexpression of AtOFP13, AtOFP15, AtOFP16 and AtOFP18 led to blunt-end siliques (class III) .\\n(#3) Class I and II AtOFPs just fell into clade C2 while class 200 bp 400 bp 600 bp 800 bp 1000 bp1200 bp 1400 bp 1600 bp.',\n","  'label': [[0, 4, 'Bad structure'],\n","   [6, 8, 'Score 4'],\n","   [6, 9, 'Topic sentence'],\n","   [33, 62, 'Paragraph Topic'],\n","   [51, 62, 'Topic attribute 1'],\n","   [154, 156, 'Score 3'],\n","   [154, 156, 'Supporting detail sentence'],\n","   [165, 221, 'Paragraph Topic'],\n","   [180, 210, 'Topic attribute 1'],\n","   [215, 221, 'Topic attribute 1'],\n","   [299, 335, 'Paragraph Topic'],\n","   [317, 334, 'Topic attribute 1'],\n","   [386, 442, 'Paragraph Topic'],\n","   [404, 441, 'Topic attribute 1'],\n","   [483, 485, 'Score 1'],\n","   [483, 485, 'Supporting detail sentence'],\n","   [502, 508, 'Topic attribute 1']],\n","  'previous_para': 'SKIP\\n(#1) The rice genome has 33 OVATE-encoding genes designated OsOFP genes.\\n(#2) These OsOFP genes are distributed unevenly across the 12 chromosomes of rice, with the highest density on chromosome 1 (eight genes), followed by chromosome 5 with seven OsOFP genes, while chromosomes 6 and 9 have no OFP genes (Fig.\\n(#3) 4A) .\\n(#4) The phylogenetic relationships of OsOFPs within rice conformed to the analysis based on the conserved OVATE region in all 13 plant genomes (Fig.\\n(#5) 5A) .\\n(#6) The OsOFP family could be divided into ten sub-groups, of which sub-group 10 was a monocot-specific clade, and sub-group 5 contained eight members accounting for a quarter of the OsOFP family in rice (Fig.\\n(#7) 5A) .\\n(#8) The genic structures of all OsOFP-coding genes were also examined.\\n(#9) Nearly all OsOFP genes are intronless, except OsOFP10, which contains a small intron (Fig.\\n(#10) 5D ).'},\n"," {'id': 36242,\n","  'text': 'SKIP\\n(#1) III corresponded to clade C1 according to our study.\\n(#2) Thus it appears that the evolution of the AtOFP family represented here is consistent with their probable patterns of functional divergence.',\n","  'label': [[0, 4, 'Too short']],\n","  'previous_para': 'SKIP\\n(#1) It is of interest that overexpression of AtOFP genes with close phylogenetic relationships was previously shown to produce similar phenotypes.\\n(#2) Plants overexpressing AtOFP1, AtOFP2, AtOFP4, AtOFP5 and AtOFP7 resulted in kidney-shaped cotyledons, and round and curled leaves (class I); overexpression of AtOFP6 and AtOFP8 caused flat, thick and cyan leaves (class II); and overexpression of AtOFP13, AtOFP15, AtOFP16 and AtOFP18 led to blunt-end siliques (class III) .\\n(#3) Class I and II AtOFPs just fell into clade C2 while class 200 bp 400 bp 600 bp 800 bp 1000 bp1200 bp 1400 bp 1600 bp.'},\n"," {'id': 36243,\n","  'text': 'SKIP\\n(#1) In the tomato genome, over a quarter of the SlOFP family genes (7\\\\/26) are located in a single gene cluster on chromosome 10, with the remaining genes dispersed across all 11 remaining chromosomes except chromosome 7 (Fig.\\n(#2) 4C) .\\n(#3) Nine members of the SlOFP family are located in sub-group 1, including five of those linked in the cluster at the end of chromosome 10 (Figs 4C and 5C).\\n(#4) To investigate further the diversity of SlOFP genes, their expression patterns in a round-fruit tomato variety were examined by quantitative real-time PCR.\\n(#5) Of the 26 SlOFP genes, 12 (SlOFP2, SlOFP3, SlOFP5, SlOFP9, SlOFP10, SlOFP18, SlOFP19, SlOFP20, SlOFP21, SlOFP23, SlOFP24 and SlOFP26) were found to be expressed in all tissues examined, while expression of SlOFP7 and SlOFP16 ws undetectable (Fig.\\n(#6) 6) .\\n(#7) One possibility is that their transcripts are present at a level below the limit of detection, or are only induced in response to certain conditions or treatments or at specific developmental stages.\\n(#8) Alternatively, SlOFP16 may be a pseudogene, which is consistent with the fact that the OVATE region of SlOFP16 does not align with that of the other OVATE protein sequences.\\n(#9) Tissue-specific expression profiles were discovered for three genes, with SlOFP4, SlOFP17 and SlOFP22 specific to stamens, sepals and pistils within the floral organs, respectively.\\n(#10) Meanwhile, SlOFP5 was most highly transcribed in the leaf, while relatively high expression of SlOFP1, SlOFP2 and SlOFP8 was detected in pistils and a high abundance of SlOFP2 and SlOFP8 transcripts was also detected in roots.\\n(#11) Preferential or tissue-specific transcription of SlOFP genes may be indicative of tissue-specific functions in plant growth and development.',\n","  'label': [[0, 4, 'Bad parse'],\n","   [6, 8, 'Score 3'],\n","   [6, 8, 'Topic sentence'],\n","   [54, 72, 'Paragraph Topic'],\n","   [194, 205, 'Topic attribute 1'],\n","   [213, 223, 'Topic attribute 1'],\n","   [233, 235, 'Score 1'],\n","   [244, 246, 'Score 3'],\n","   [244, 246, 'Supporting detail sentence'],\n","   [268, 280, 'Paragraph Topic'],\n","   [369, 379, 'Topic attribute 1'],\n","   [402, 404, 'Score 3'],\n","   [402, 404, 'Supporting detail sentence'],\n","   [446, 457, 'Paragraph Topic'],\n","   [563, 565, 'Score 3'],\n","   [563, 565, 'Supporting detail sentence'],\n","   [577, 588, 'Link 1'],\n","   [577, 588, 'Paragraph Topic'],\n","   [594, 680, 'Paragraph Topic'],\n","   [680, 699, 'Paragraph Topic'],\n","   [773, 791, 'Paragraph Topic'],\n","   [815, 817, 'Score 1'],\n","   [825, 827, 'Score 3'],\n","   [825, 827, 'Supporting detail sentence'],\n","   [853, 870, 'Link 1'],\n","   [1030, 1032, 'Score 3'],\n","   [1030, 1032, 'Supporting detail sentence'],\n","   [1049, 1056, 'Paragraph Topic'],\n","   [1137, 1144, 'Paragraph Topic'],\n","   [1209, 1211, 'Score 3'],\n","   [1209, 1211, 'Supporting detail sentence'],\n","   [1287, 1314, 'Paragraph Topic'],\n","   [1396, 1399, 'Supporting detail sentence'],\n","   [1396, 1400, 'Score 4'],\n","   [1412, 1418, 'Paragraph Topic'],\n","   [1496, 1510, 'Paragraph Topic'],\n","   [1515, 1521, 'Paragraph Topic'],\n","   [1570, 1588, 'Paragraph Topic'],\n","   [1629, 1632, 'Score 3'],\n","   [1629, 1633, 'Conclusion sentence'],\n","   [1683, 1694, 'Paragraph Topic']],\n","  'previous_para': 'SKIP\\n(#1) III corresponded to clade C1 according to our study.\\n(#2) Thus it appears that the evolution of the AtOFP family represented here is consistent with their probable patterns of functional divergence.'},\n"," {'id': 36244,\n","  'text': 'SKIP\\n(#1) From the phylogenetic analysis of SlOFPs, nine pairs of paralogues could be identified, eight of which were well supported by bootstrap analysis (Fig.\\n(#2) 5C ).\\n(#3) Their phylogenetic relationships and expression patterns revealed two fates of the duplicated paralogues, i.e.\\n(#4) retention -retention (RR) indicating that two paralogues retain the same original expression pattern or function, and retention -divergence (RD) indicating that one paralogue retains but the other diverges in expression pattern or function.\\n(#5) Four pairs of paralogues (SlOFP8 and SlOFP13, SlOFP19 and SlOFP20, SlOFP18 and SlOFP21, and SlOFP2 and SlOFP9) were consistent with the RR mode, as they exhibited the same expression patterns.The other four pairs of paralogues (SlOFP17 and SlOFP 24, SlOFP1 and SlOFP 14, SlOFP6 and SlOFP10, and SlOFP25 and SlOFP26) were considered to be RD mode owing to their diverged expression patterns (Fig.\\n(#6) 6 ).\\n(#7) For example, SlOFP24 was found to be ubiquitously expressed in all tissues, whereas the transcript of its most closely related paralogue SlOFP17 was restricted to the sepal (Fig.\\n(#8) 6) .\\n(#9) Both subfunctionalization and neofunctionalization might follow the divergence.',\n","  'label': [[0, 4, 'Bad parse'],\n","   [0, 4, 'Bad structure'],\n","   [6, 8, 'Topic sentence'],\n","   [6, 9, 'Score 3'],\n","   [19, 50, 'Paragraph Topic'],\n","   [162, 164, 'Score 1'],\n","   [173, 175, 'Score 3'],\n","   [173, 176, 'Supporting detail sentence'],\n","   [177, 209, 'Paragraph Topic'],\n","   [289, 291, 'Supporting detail sentence'],\n","   [289, 292, 'Score 1'],\n","   [340, 349, 'Topic attribute 1'],\n","   [458, 467, 'Topic attribute 1'],\n","   [535, 537, 'Score 3'],\n","   [535, 538, 'Supporting detail sentence'],\n","   [553, 563, 'Topic attribute 1'],\n","   [565, 650, 'Paragraph Topic'],\n","   [755, 766, 'Topic attribute 1'],\n","   [767, 853, 'Paragraph Topic'],\n","   [935, 939, 'Score 1'],\n","   [946, 948, 'Score 3'],\n","   [946, 948, 'Supporting detail sentence'],\n","   [963, 971, 'Paragraph Topic'],\n","   [1077, 1086, 'Topic attribute 1'],\n","   [1087, 1094, 'Paragraph Topic'],\n","   [1129, 1133, 'Score 1'],\n","   [1140, 1142, 'Score 3'],\n","   [1140, 1142, 'Supporting detail sentence']],\n","  'previous_para': 'SKIP\\n(#1) In the tomato genome, over a quarter of the SlOFP family genes (7\\\\/26) are located in a single gene cluster on chromosome 10, with the remaining genes dispersed across all 11 remaining chromosomes except chromosome 7 (Fig.\\n(#2) 4C) .\\n(#3) Nine members of the SlOFP family are located in sub-group 1, including five of those linked in the cluster at the end of chromosome 10 (Figs 4C and 5C).\\n(#4) To investigate further the diversity of SlOFP genes, their expression patterns in a round-fruit tomato variety were examined by quantitative real-time PCR.\\n(#5) Of the 26 SlOFP genes, 12 (SlOFP2, SlOFP3, SlOFP5, SlOFP9, SlOFP10, SlOFP18, SlOFP19, SlOFP20, SlOFP21, SlOFP23, SlOFP24 and SlOFP26) were found to be expressed in all tissues examined, while expression of SlOFP7 and SlOFP16 ws undetectable (Fig.\\n(#6) 6) .\\n(#7) One possibility is that their transcripts are present at a level below the limit of detection, or are only induced in response to certain conditions or treatments or at specific developmental stages.\\n(#8) Alternatively, SlOFP16 may be a pseudogene, which is consistent with the fact that the OVATE region of SlOFP16 does not align with that of the other OVATE protein sequences.\\n(#9) Tissue-specific expression profiles were discovered for three genes, with SlOFP4, SlOFP17 and SlOFP22 specific to stamens, sepals and pistils within the floral organs, respectively.\\n(#10) Meanwhile, SlOFP5 was most highly transcribed in the leaf, while relatively high expression of SlOFP1, SlOFP2 and SlOFP8 was detected in pistils and a high abundance of SlOFP2 and SlOFP8 transcripts was also detected in roots.\\n(#11) Preferential or tissue-specific transcription of SlOFP genes may be indicative of tissue-specific functions in plant growth and development.'},\n"," {'id': 36245,\n","  'text': 'SKIP\\n(#1) Duplication and OVATE gene family expansion in arabidopsis, rice and tomato Plant genomes contain a higher proportion of duplicated genes than most other eukaryotic genomes and this has been argued to be a robust evolutionary force (Lockton and Gaut, 2005 ; Shan al., 2007) .\\n(#2) Gene duplication is speculated to provide the raw genetic materials for the phenotypic or functional novelty necessary for adaptive evolution (Flagel and Wendel, 2009; Jiao et al., 2011) .\\n(#3) Segmental duplication, tandem duplication and retrotransposition or other transposition events are three major mechanisms for gene duplication (Kong et al., 2007) .\\n(#4) Analysis of 50 large gene families in arabidopsis combining information about genomic segmental duplications, gene family phylogeny and gene positions revealed that some gene families expanded mainly through tandem duplication, while some were oriented towards segmental duplication (Cannon et al., 2004) .\\n(#5) Based on the chromosomal location and phylogenetic relationships of OVATE family genes, we analysed the relative contributions of tandem duplication and segmental duplication to OVATE family expansion in arabidopsis, rice and tomato.\\n(#6) Of the 19 AtOFP genes, eight (42 .\\n(#7) 1 %) and 12 (63 .\\n(#8) 2 %) appear to have arisen by tandem duplication and segmental duplication (Figs 4B and 5B), respectively.\\n(#9) Within the rice OsOFP family, eight (24 .\\n(#10) 2 %) have resulted from tandem duplication, whereas the majority (24\\\\/33; 72 .\\n(#11) 7 %) were present within segmental duplication blocks (Figs 4A and 5A) .\\n(#12) In terms of the SlOFP genes, ten (38 .\\n(#13) 5 %) were tandemly duplicated and nine (34 .\\n(#14) 6 %) were located within segmental duplication blocks ( Figs 4C and 5C ).\\n(#15) Therefore, tandem duplication and segmental duplication appear to play a principle role in SlOFP gene family expansion in all of the three sampled plant genomes, with segmental duplication particularly important for expansion of the arabidopsis AtOFP and rice OsOFP gene family (Fig.\\n(#16) 7A, B) .\\n(#17) This is similar to the observation in other large gene families such as the bHLH (basic helix -loop -helix) gene family in rice .\\n(#18) Also, the largest tandem-arrayed SlOFP gene cluster, exhibiting a single-lineage expansion, on tomato chromosome 10 arose from a tandem duplication event (Fig.\\n(#19) 7C) .\\n(#20) It is reported that stress-responsive gene sets are enriched for tandemly duplicated genes, thus tandemly duplicated genes involved in stress response are suggested to be preferably retained (Rizzon et al., 2006; Hanada et al., 2008) .\\n(#21) To better understand the evolutionary history of the OVATE family in arabidopsis, rice and tomato, we estimated the timing of segmental duplication events using values for Ks as the proxy for time (Tables 2 -4) .\\n(#22) Ks values of AtOFP parologues centre around two ranges, from 0 .\\n(#23) 70 to 0 .\\n(#24) 84 and from 1 .\\n(#25) 51 to 1 .\\n(#26) 82.\\n(#27) This suggests that two major duplication events occurred in arabidopsis, one around 23 .\\n(#28) 33-28 .\\n(#29) 00 million years ago (Mya) and a second around 50 .\\n(#30) 33-60 .\\n(#31) 67 Mya.\\n(#32) These dates are approximately in line with the two WGD events (a and b) reported for arabidopsis, subsequent to the divergence of arabidopsis and C.\\n(#33) papaya around 70 Mya (Bowers et al., 2003; Lockton and Gaut, 2005; Ming et al., 2008; Tang et al., 2008; Abrouk et al., 2010; Jiao et al., 2011) .\\n(#34) These results hint that the duplications in AtOFP sequences may be a consequence of the two WGDs, or of isolated, local segmental duplication events that occurred within those two periods.\\n(#35) For the rice OsOFP genes, most of the segmental duplication events were estimated to occur between 50 and 70 Mya, consistent with the r WGD event pre-dating the diversification of major cereal clades (Paterson et al., 2004; Wang et al., 2005; Salse et al., 2008) .\\n(#36) The duplication time of paralogous SlOFP genes via segmental duplication was estimated to occur between 29 and 78 .\\n(#37) 33 Mya, which may have involved the recent triplication event (52 -91 Mya) shared by both tomato and potato in the Solanaceae, preceding their divergence about 7 .\\n(#38) 3 Mya (Tomato Genome Consortium, 2012).',\n","  'label': [[0, 4, 'Bad parse'],\n","   [6, 8, 'Score 3'],\n","   [6, 9, 'Topic sentence'],\n","   [10, 53, 'Paragraph Topic'],\n","   [287, 289, 'Score 3'],\n","   [287, 289, 'Supporting detail sentence'],\n","   [291, 307, 'Paragraph Topic'],\n","   [481, 483, 'Score 4'],\n","   [481, 483, 'Supporting detail sentence'],\n","   [485, 549, 'Topic attribute 1'],\n","   [651, 653, 'Score 3'],\n","   [651, 653, 'Supporting detail sentence'],\n","   [741, 764, 'Topic attribute 1'],\n","   [863, 881, 'Topic attribute 1'],\n","   [916, 937, 'Topic attribute 1'],\n","   [963, 965, 'Score 4'],\n","   [963, 965, 'Supporting detail sentence'],\n","   [1035, 1053, 'Paragraph Topic'],\n","   [1098, 1115, 'Topic attribute 1'],\n","   [1120, 1141, 'Topic attribute 1'],\n","   [1145, 1168, 'Paragraph Topic'],\n","   [1202, 1204, 'Score 1'],\n","   [1242, 1244, 'Score 1'],\n","   [1265, 1268, 'Score 1'],\n","   [1299, 1317, 'Topic attribute 1'],\n","   [1322, 1344, 'Topic attribute 1'],\n","   [1377, 1380, 'Score 1'],\n","   [1424, 1427, 'Score 1'],\n","   [1453, 1472, 'Topic attribute 1'],\n","   [1508, 1511, 'Score 1'],\n","   [1538, 1559, 'Topic attribute 1'],\n","   [1587, 1590, 'Score 1'],\n","   [1632, 1635, 'Score 1'],\n","   [1647, 1667, 'Topic attribute 1'],\n","   [1683, 1686, 'Score 1'],\n","   [1713, 1734, 'Topic attribute 1'],\n","   [1763, 1766, 'Conclusion sentence'],\n","   [1763, 1766, 'Score 3'],\n","   [1779, 1797, 'Topic attribute 1'],\n","   [1802, 1823, 'Topic attribute 1'],\n","   [1935, 1955, 'Topic attribute 1'],\n","   [2053, 2056, 'Score 1'],\n","   [2068, 2071, 'Supporting detail sentence'],\n","   [2069, 2071, 'Score 3'],\n","   [2204, 2207, 'Score 3'],\n","   [2204, 2207, 'Supporting detail sentence'],\n","   [2370, 2373, 'Score 1'],\n","   [2382, 2385, 'Score 3'],\n","   [2382, 2386, 'Supporting detail sentence'],\n","   [2452, 2471, 'Topic attribute 1'],\n","   [2485, 2504, 'Topic attribute 1'],\n","   [2624, 2627, 'Score 3'],\n","   [2624, 2627, 'Supporting detail sentence'],\n","   [2844, 2846, 'Score 1'],\n","   [2914, 2918, 'Score 1'],\n","   [2930, 2933, 'Score 1'],\n","   [2952, 2955, 'Score 1'],\n","   [2968, 2971, 'Score 1'],\n","   [2978, 2981, 'Score 1'],\n","   [3073, 3076, 'Score 1'],\n","   [3087, 3090, 'Score 1'],\n","   [3145, 3148, 'Score 1'],\n","   [3159, 3162, 'Score 1'],\n","   [3173, 3176, 'Supporting detail sentence'],\n","   [3173, 3177, 'Score 1'],\n","   [3328, 3332, 'Score 1'],\n","   [3481, 3484, 'Supporting detail sentence'],\n","   [3606, 3627, 'Topic attribute 1'],\n","   [3676, 3679, 'Supporting detail sentence'],\n","   [3719, 3740, 'Topic attribute 1'],\n","   [3947, 3950, 'Score 1'],\n","   [3947, 3950, 'Supporting detail sentence'],\n","   [4003, 4024, 'Topic attribute 1'],\n","   [4069, 4072, 'Score 1'],\n","   [4069, 4072, 'Supporting detail sentence'],\n","   [4239, 4242, 'Score 1']],\n","  'previous_para': 'SKIP\\n(#1) From the phylogenetic analysis of SlOFPs, nine pairs of paralogues could be identified, eight of which were well supported by bootstrap analysis (Fig.\\n(#2) 5C ).\\n(#3) Their phylogenetic relationships and expression patterns revealed two fates of the duplicated paralogues, i.e.\\n(#4) retention -retention (RR) indicating that two paralogues retain the same original expression pattern or function, and retention -divergence (RD) indicating that one paralogue retains but the other diverges in expression pattern or function.\\n(#5) Four pairs of paralogues (SlOFP8 and SlOFP13, SlOFP19 and SlOFP20, SlOFP18 and SlOFP21, and SlOFP2 and SlOFP9) were consistent with the RR mode, as they exhibited the same expression patterns.The other four pairs of paralogues (SlOFP17 and SlOFP 24, SlOFP1 and SlOFP 14, SlOFP6 and SlOFP10, and SlOFP25 and SlOFP26) were considered to be RD mode owing to their diverged expression patterns (Fig.\\n(#6) 6 ).\\n(#7) For example, SlOFP24 was found to be ubiquitously expressed in all tissues, whereas the transcript of its most closely related paralogue SlOFP17 was restricted to the sepal (Fig.\\n(#8) 6) .\\n(#9) Both subfunctionalization and neofunctionalization might follow the divergence.'},\n"," {'id': 36246,\n","  'text': 'SKIP\\n(#1) In this study, we performed the first comparative genomic analysis of the OVATE protein family, a recently discovered plant-specific gene regulatory family in land plants.\\n(#2) OVATE family proteins are present in all major lineages of land plants including the early-diverged species.\\n(#3) Our phylogenetic analysis of 13 available plant genomes spanning major evolutionary lineages defined 11 sub-groups of OVATE family proteins in angiosperms.\\n(#4) Two different mechanisms, namely conserved evolution and divergent expansion, are proposed to be involved in OVATE family evolution in plants.\\n(#5) Detailed characterization of the AtOFP family in arabidopsis, the OsOFP family in rice and the SlOFP family in tomato provided a deeper understanding of the evolutionary framework and revealed a principle role for tandem duplication and segmental duplication in expansion of the OVATE gene family.\\n(#6) This study has established a solid base for subsequent functional genomics studies on this important gene family in plants, which has been poorly characterized to date.',\n","  'label': [[5, 8, 'Score 5'],\n","   [5, 9, 'Topic sentence'],\n","   [48, 104, 'Paragraph Topic'],\n","   [84, 104, 'Topic attribute 1'],\n","   [183, 185, 'Score 4'],\n","   [183, 185, 'Supporting detail sentence'],\n","   [187, 208, 'Topic attribute 1'],\n","   [297, 299, 'Score 4'],\n","   [297, 299, 'Supporting detail sentence'],\n","   [301, 326, 'Paragraph Topic'],\n","   [419, 440, 'Topic attribute 1'],\n","   [458, 460, 'Score 3'],\n","   [458, 460, 'Supporting detail sentence'],\n","   [495, 514, 'Topic attribute 2'],\n","   [519, 538, 'Topic attribute 1'],\n","   [571, 583, 'Topic attribute 1'],\n","   [606, 608, 'Score 3'],\n","   [606, 608, 'Supporting detail sentence'],\n","   [824, 842, 'Topic attribute 3'],\n","   [847, 868, 'Topic attribute 4'],\n","   [885, 906, 'Topic attribute 1'],\n","   [909, 911, 'Conclusion sentence'],\n","   [909, 911, 'Score 5'],\n","   [913, 923, 'Paragraph Topic'],\n","   [998, 1025, 'Topic attribute 1']],\n","  'previous_para': 'SKIP\\n(#1) Duplication and OVATE gene family expansion in arabidopsis, rice and tomato Plant genomes contain a higher proportion of duplicated genes than most other eukaryotic genomes and this has been argued to be a robust evolutionary force (Lockton and Gaut, 2005 ; Shan al., 2007) .\\n(#2) Gene duplication is speculated to provide the raw genetic materials for the phenotypic or functional novelty necessary for adaptive evolution (Flagel and Wendel, 2009; Jiao et al., 2011) .\\n(#3) Segmental duplication, tandem duplication and retrotransposition or other transposition events are three major mechanisms for gene duplication (Kong et al., 2007) .\\n(#4) Analysis of 50 large gene families in arabidopsis combining information about genomic segmental duplications, gene family phylogeny and gene positions revealed that some gene families expanded mainly through tandem duplication, while some were oriented towards segmental duplication (Cannon et al., 2004) .\\n(#5) Based on the chromosomal location and phylogenetic relationships of OVATE family genes, we analysed the relative contributions of tandem duplication and segmental duplication to OVATE family expansion in arabidopsis, rice and tomato.\\n(#6) Of the 19 AtOFP genes, eight (42 .\\n(#7) 1 %) and 12 (63 .\\n(#8) 2 %) appear to have arisen by tandem duplication and segmental duplication (Figs 4B and 5B), respectively.\\n(#9) Within the rice OsOFP family, eight (24 .\\n(#10) 2 %) have resulted from tandem duplication, whereas the majority (24\\\\/33; 72 .\\n(#11) 7 %) were present within segmental duplication blocks (Figs 4A and 5A) .\\n(#12) In terms of the SlOFP genes, ten (38 .\\n(#13) 5 %) were tandemly duplicated and nine (34 .\\n(#14) 6 %) were located within segmental duplication blocks ( Figs 4C and 5C ).\\n(#15) Therefore, tandem duplication and segmental duplication appear to play a principle role in SlOFP gene family expansion in all of the three sampled plant genomes, with segmental duplication particularly important for expansion of the arabidopsis AtOFP and rice OsOFP gene family (Fig.\\n(#16) 7A, B) .\\n(#17) This is similar to the observation in other large gene families such as the bHLH (basic helix -loop -helix) gene family in rice .\\n(#18) Also, the largest tandem-arrayed SlOFP gene cluster, exhibiting a single-lineage expansion, on tomato chromosome 10 arose from a tandem duplication event (Fig.\\n(#19) 7C) .\\n(#20) It is reported that stress-responsive gene sets are enriched for tandemly duplicated genes, thus tandemly duplicated genes involved in stress response are suggested to be preferably retained (Rizzon et al., 2006; Hanada et al., 2008) .\\n(#21) To better understand the evolutionary history of the OVATE family in arabidopsis, rice and tomato, we estimated the timing of segmental duplication events using values for Ks as the proxy for time (Tables 2 -4) .\\n(#22) Ks values of AtOFP parologues centre around two ranges, from 0 .\\n(#23) 70 to 0 .\\n(#24) 84 and from 1 .\\n(#25) 51 to 1 .\\n(#26) 82.\\n(#27) This suggests that two major duplication events occurred in arabidopsis, one around 23 .\\n(#28) 33-28 .\\n(#29) 00 million years ago (Mya) and a second around 50 .\\n(#30) 33-60 .\\n(#31) 67 Mya.\\n(#32) These dates are approximately in line with the two WGD events (a and b) reported for arabidopsis, subsequent to the divergence of arabidopsis and C.\\n(#33) papaya around 70 Mya (Bowers et al., 2003; Lockton and Gaut, 2005; Ming et al., 2008; Tang et al., 2008; Abrouk et al., 2010; Jiao et al., 2011) .\\n(#34) These results hint that the duplications in AtOFP sequences may be a consequence of the two WGDs, or of isolated, local segmental duplication events that occurred within those two periods.\\n(#35) For the rice OsOFP genes, most of the segmental duplication events were estimated to occur between 50 and 70 Mya, consistent with the r WGD event pre-dating the diversification of major cereal clades (Paterson et al., 2004; Wang et al., 2005; Salse et al., 2008) .\\n(#36) The duplication time of paralogous SlOFP genes via segmental duplication was estimated to occur between 29 and 78 .\\n(#37) 33 Mya, which may have involved the recent triplication event (52 -91 Mya) shared by both tomato and potato in the Solanaceae, preceding their divergence about 7 .\\n(#38) 3 Mya (Tomato Genome Consortium, 2012).'}]"]},"metadata":{},"execution_count":8}]},{"cell_type":"code","source":["loaded_dict[\"paper_1\"]['abstract']"],"metadata":{"colab":{"base_uri":"https://localhost:8080/","height":191},"id":"2RUdQ4bNNuwa","executionInfo":{"status":"ok","timestamp":1712916311502,"user_tz":-120,"elapsed":5,"user":{"displayName":"Anna Kiepura","userId":"06512247374108941998"}},"outputId":"ae4f6afc-b374-4670-c8d5-8a691deb381e"},"execution_count":9,"outputs":[{"output_type":"execute_result","data":{"text/plain":["'Background and aims: The OVATE gene encodes a nuclear-localized regulatory protein belonging to a distinct family of plant-specific proteins known as the OVATE family proteins (OFPs). OVATE was first identified as a key regulator of fruit shape in tomato, with nonsense mutants displaying pear-shaped fruits. However, the role of OFPs in plant development has been poorly characterized. Methods: Public databases were searched and a total of 265 putative OVATE protein sequences were identified from 13 sequenced plant genomes that represent the major evolutionary lineages of land plants. A phylogenetic analysis was conducted based on the alignment of the conserved OVATE domain from these 13 selected plant genomes. The expression patterns of tomato SlOFP genes were analysed via quantitative real-time PCR. The pattern of OVATE gene duplication resulting in the expansion of the gene family was determined in arabidopsis, rice and tomato. Key results: Genes for OFPs were found to be present in all the sampled land plant genomes, including the early-diverged lineages, mosses and lycophytes. Phylogenetic analysis based on the amino acid sequences of the conserved OVATE domain defined 11 sub-groups of OFPs in angiosperms. Different evolutionary mechanisms are proposed for OVATE family evolution, namely conserved evolution and divergent expansion. Characterization of the AtOFP family in arabidopsis, the OsOFP family in rice and the SlOFP family in tomato provided further details regarding the evolutionary framework and revealed a major contribution of tandem and segmental duplications towards expansion of the OVATE gene family. Conclusions: This first genome-wide survey on OFPs provides new insights into the evolution of the OVATE protein family and establishes a solid base for future functional genomics studies on this important but poorly characterized regulatory protein family in plants.'"],"application/vnd.google.colaboratory.intrinsic+json":{"type":"string"}},"metadata":{},"execution_count":9}]},{"cell_type":"markdown","source":["Below, we concatenate all paragraphs in the dataset into 1 list"],"metadata":{"id":"iKfmvo5iMvYY"}},{"cell_type":"code","source":["paras_list = []\n","\n","for a in list(loaded_dict.keys()):\n","  paper = loaded_dict[a]\n","  for b in paper['paper']:\n","    paras_list.append(b)"],"metadata":{"id":"1h74UZeELYci","executionInfo":{"status":"ok","timestamp":1712916314452,"user_tz":-120,"elapsed":3,"user":{"displayName":"Anna Kiepura","userId":"06512247374108941998"}}},"execution_count":10,"outputs":[]},{"cell_type":"markdown","source":["For each paragraph, we have the paragraph id, the paragraph text, all labels placed on the paragraph by the annotator, and the previous paragraph from the corresponding paper (expect for the very first paragraphs of each paper)"],"metadata":{"id":"5kgEsQnqM62l"}},{"cell_type":"code","source":["paras_list[1]"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"fX-KC8-8MN0f","executionInfo":{"status":"ok","timestamp":1712916316549,"user_tz":-120,"elapsed":5,"user":{"displayName":"Anna Kiepura","userId":"06512247374108941998"}},"outputId":"4e15556b-e770-496f-c185-d73922bbab82"},"execution_count":11,"outputs":[{"output_type":"execute_result","data":{"text/plain":["{'id': 36212,\n"," 'text': 'SKIP\\n(#1) To date, OFPs have been primarily characterized in arabidopsis (AtOFPs) and demonstrated to regulate plant growth and development (Hackbusch et al., 2005; Pagnussat et al., 2007; Wang et al., 2007 Wang et al., , 2010 Li et al., 2011) .\\n(#2) AtOFPs were shown to have close functional interactions with three amino acid loop extension (TALE) homeodomain proteins, and AtOFP1 and AtOFP5 regulate the sub-cellular localization of TALE homeoproteins (Hackbusch et al., 2005) .\\n(#3) The plant TALE proteins are a conserved superclass of homeodomain proteins characterized by an extension of three amino acids between helices 1 and 2 of the homeodomain (Bertolino et al., 1995) , and comprise two sub-classes called the KNOTTED-like homeobox (KNOX) and BEL1-like homeodomain (BELL) proteins Tsiantis, 2009, 2010; Hamant and Pautot, 2010) .\\n(#4) Furthermore, it has been well documented that interactions between KNOX and BELL proteins result in heterodimers regulating plant development in a connected and complex network (Bellaoui et al., 2001; Smith et al., 2002; Smith and Hake, 2003; Chen et al., 2004; Hackbusch et al., 2005; Cole et al., 2006) .\\n(#5) AtOFP1 has been reported to function as an active transcriptional repressor of AtGA20ox1 in the gibberellin (GA) biosynthesis pathway, suppressing cell elongation (Wang et al., 2007) .\\n(#6) A recent study also indicated that AtOFP1 interacts with AtKu70, a protein involved in DNA repair through the non-homologous end-joining pathway .\\n(#7) Similar to AtOFP1, AtOFP4 acts as a transcriptional repressor and has been proposed to form a functional complex with KNAT7, one of four class II arabidopsis KNOTTED1-like Arabidopsis thaliana (KNAT) members (Hake et al., 2004; Hamant and Pautot, 2010) , in regulating secondary cell wall formation (Li et al., 2011) .\\n(#8) AtOFP5, reported to interact with both BLH1 and KNAT3, which belong to BELL and KNOX sub-classes of TALE homeodomain proteins, respectively (Hackbusch et al., 2005) , can act as regulator of the BELL-KNOX TALE complex involved in normal embryo sac development in arabidopsis (Pagnussat et al., 2007) .\\n(#9) Recently, a genome-wide analysis of AtOFPs revealed conserved functions as transcriptional repressors, with overexpression leading to a number of abnormal phenotypes, implying novel roles in regulating plant growth and development .',\n"," 'label': [[0, 4, 'Bad structure'],\n","  [5, 9, 'Score 4'],\n","  [5, 9, 'Topic sentence'],\n","  [19, 23, 'Paragraph Topic'],\n","  [74, 80, 'Paragraph Topic'],\n","  [246, 250, 'Score 2'],\n","  [246, 250, 'Supporting detail sentence'],\n","  [251, 257, 'Paragraph Topic'],\n","  [345, 349, 'Topic attribute 1'],\n","  [377, 383, 'Topic attribute 2'],\n","  [388, 394, 'Topic attribute 4'],\n","  [436, 441, 'Topic attribute 1'],\n","  [483, 487, 'Score 3'],\n","  [483, 487, 'Supporting detail sentence'],\n","  [498, 503, 'Topic attribute 1'],\n","  [747, 751, 'Topic attribute 3'],\n","  [757, 785, 'Topic attribute 6'],\n","  [844, 848, 'Supporting detail sentence'],\n","  [845, 848, 'Score 4'],\n","  [916, 920, 'Topic attribute 3'],\n","  [925, 938, 'Topic attribute 6'],\n","  [1156, 1160, 'Supporting detail sentence'],\n","  [1157, 1160, 'Score 3'],\n","  [1161, 1167, 'Topic attribute 2'],\n","  [1346, 1350, 'Score 4'],\n","  [1346, 1350, 'Supporting detail sentence'],\n","  [1386, 1392, 'Topic attribute 2'],\n","  [1498, 1502, 'Score 2'],\n","  [1498, 1502, 'Supporting detail sentence'],\n","  [1514, 1520, 'Topic attribute 2'],\n","  [1621, 1626, 'Topic attribute 5'],\n","  [1697, 1701, 'Topic attribute 5'],\n","  [1822, 1826, 'Score 3'],\n","  [1822, 1826, 'Supporting detail sentence'],\n","  [1827, 1833, 'Topic attribute 4'],\n","  [1875, 1880, 'Topic attribute 5'],\n","  [1898, 1902, 'Topic attribute 6'],\n","  [1907, 1911, 'Topic attribute 3'],\n","  [1927, 1931, 'Topic attribute 1'],\n","  [2022, 2026, 'Topic attribute 6'],\n","  [2027, 2031, 'Topic attribute 3'],\n","  [2032, 2036, 'Topic attribute 1'],\n","  [2129, 2133, 'Score 3'],\n","  [2129, 2133, 'Supporting detail sentence'],\n","  [2170, 2176, 'Paragraph Topic']],\n"," 'previous_para': 'SKIP\\nPaper title: Phylogenetic analyses provide the first insights into the evolution of OVATE family proteins in land plants. \\n\\n(#1) The OVATE gene was first identified as an important regulator of fruit shape in tomato, in which a naturally occurring premature stop codon in OVATE results in pear-shaped fruit with longitudinal elongation and neck constriction (Liu et al., 2002) .\\n(#2) This revealed a previously uncharacterized class of regulatory genes in plant development, which encode proteins with a conserved 70 amino acid C-terminal domain.\\n(#3) This domain was designated as the OVATE domain, also known as DUF623 (Domain of Unknown Function 623), and proteins containing this domain were designated OVATE family proteins (OFPs), which are found exclusively in plants (Hackbusch et al., 2005; Wang et al., 2007 Wang et al., , 2011 .'}"]},"metadata":{},"execution_count":11}]},{"cell_type":"code","source":["paras_list[1]['text']"],"metadata":{"colab":{"base_uri":"https://localhost:8080/","height":191},"id":"JwAkqDmfOI-c","executionInfo":{"status":"ok","timestamp":1712916317620,"user_tz":-120,"elapsed":7,"user":{"displayName":"Anna Kiepura","userId":"06512247374108941998"}},"outputId":"24bc9887-e61b-4e1f-ca16-59306b8051d1"},"execution_count":12,"outputs":[{"output_type":"execute_result","data":{"text/plain":["'SKIP\\n(#1) To date, OFPs have been primarily characterized in arabidopsis (AtOFPs) and demonstrated to regulate plant growth and development (Hackbusch et al., 2005; Pagnussat et al., 2007; Wang et al., 2007 Wang et al., , 2010 Li et al., 2011) .\\n(#2) AtOFPs were shown to have close functional interactions with three amino acid loop extension (TALE) homeodomain proteins, and AtOFP1 and AtOFP5 regulate the sub-cellular localization of TALE homeoproteins (Hackbusch et al., 2005) .\\n(#3) The plant TALE proteins are a conserved superclass of homeodomain proteins characterized by an extension of three amino acids between helices 1 and 2 of the homeodomain (Bertolino et al., 1995) , and comprise two sub-classes called the KNOTTED-like homeobox (KNOX) and BEL1-like homeodomain (BELL) proteins Tsiantis, 2009, 2010; Hamant and Pautot, 2010) .\\n(#4) Furthermore, it has been well documented that interactions between KNOX and BELL proteins result in heterodimers regulating plant development in a connected and complex network (Bellaoui et al., 2001; Smith et al., 2002; Smith and Hake, 2003; Chen et al., 2004; Hackbusch et al., 2005; Cole et al., 2006) .\\n(#5) AtOFP1 has been reported to function as an active transcriptional repressor of AtGA20ox1 in the gibberellin (GA) biosynthesis pathway, suppressing cell elongation (Wang et al., 2007) .\\n(#6) A recent study also indicated that AtOFP1 interacts with AtKu70, a protein involved in DNA repair through the non-homologous end-joining pathway .\\n(#7) Similar to AtOFP1, AtOFP4 acts as a transcriptional repressor and has been proposed to form a functional complex with KNAT7, one of four class II arabidopsis KNOTTED1-like Arabidopsis thaliana (KNAT) members (Hake et al., 2004; Hamant and Pautot, 2010) , in regulating secondary cell wall formation (Li et al., 2011) .\\n(#8) AtOFP5, reported to interact with both BLH1 and KNAT3, which belong to BELL and KNOX sub-classes of TALE homeodomain proteins, respectively (Hackbusch et al., 2005) , can act as regulator of the BELL-KNOX TALE complex involved in normal embryo sac development in arabidopsis (Pagnussat et al., 2007) .\\n(#9) Recently, a genome-wide analysis of AtOFPs revealed conserved functions as transcriptional repressors, with overexpression leading to a number of abnormal phenotypes, implying novel roles in regulating plant growth and development .'"],"application/vnd.google.colaboratory.intrinsic+json":{"type":"string"}},"metadata":{},"execution_count":12}]},{"cell_type":"code","source":["paras_list[1]['label']"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"8IiPrjcjOM5w","executionInfo":{"status":"ok","timestamp":1712916321599,"user_tz":-120,"elapsed":341,"user":{"displayName":"Anna Kiepura","userId":"06512247374108941998"}},"outputId":"3b76c0f5-a878-43e9-e123-ce3ec21be1f9"},"execution_count":13,"outputs":[{"output_type":"execute_result","data":{"text/plain":["[[0, 4, 'Bad structure'],\n"," [5, 9, 'Score 4'],\n"," [5, 9, 'Topic sentence'],\n"," [19, 23, 'Paragraph Topic'],\n"," [74, 80, 'Paragraph Topic'],\n"," [246, 250, 'Score 2'],\n"," [246, 250, 'Supporting detail sentence'],\n"," [251, 257, 'Paragraph Topic'],\n"," [345, 349, 'Topic attribute 1'],\n"," [377, 383, 'Topic attribute 2'],\n"," [388, 394, 'Topic attribute 4'],\n"," [436, 441, 'Topic attribute 1'],\n"," [483, 487, 'Score 3'],\n"," [483, 487, 'Supporting detail sentence'],\n"," [498, 503, 'Topic attribute 1'],\n"," [747, 751, 'Topic attribute 3'],\n"," [757, 785, 'Topic attribute 6'],\n"," [844, 848, 'Supporting detail sentence'],\n"," [845, 848, 'Score 4'],\n"," [916, 920, 'Topic attribute 3'],\n"," [925, 938, 'Topic attribute 6'],\n"," [1156, 1160, 'Supporting detail sentence'],\n"," [1157, 1160, 'Score 3'],\n"," [1161, 1167, 'Topic attribute 2'],\n"," [1346, 1350, 'Score 4'],\n"," [1346, 1350, 'Supporting detail sentence'],\n"," [1386, 1392, 'Topic attribute 2'],\n"," [1498, 1502, 'Score 2'],\n"," [1498, 1502, 'Supporting detail sentence'],\n"," [1514, 1520, 'Topic attribute 2'],\n"," [1621, 1626, 'Topic attribute 5'],\n"," [1697, 1701, 'Topic attribute 5'],\n"," [1822, 1826, 'Score 3'],\n"," [1822, 1826, 'Supporting detail sentence'],\n"," [1827, 1833, 'Topic attribute 4'],\n"," [1875, 1880, 'Topic attribute 5'],\n"," [1898, 1902, 'Topic attribute 6'],\n"," [1907, 1911, 'Topic attribute 3'],\n"," [1927, 1931, 'Topic attribute 1'],\n"," [2022, 2026, 'Topic attribute 6'],\n"," [2027, 2031, 'Topic attribute 3'],\n"," [2032, 2036, 'Topic attribute 1'],\n"," [2129, 2133, 'Score 3'],\n"," [2129, 2133, 'Supporting detail sentence'],\n"," [2170, 2176, 'Paragraph Topic']]"]},"metadata":{},"execution_count":13}]},{"cell_type":"code","source":["paras_list[1]['previous_para']"],"metadata":{"colab":{"base_uri":"https://localhost:8080/","height":174},"id":"p5Zw-w8qOQj3","executionInfo":{"status":"ok","timestamp":1712916327030,"user_tz":-120,"elapsed":269,"user":{"displayName":"Anna Kiepura","userId":"06512247374108941998"}},"outputId":"04a2be35-386d-4d35-dac5-8071eb9b4217"},"execution_count":14,"outputs":[{"output_type":"execute_result","data":{"text/plain":["'SKIP\\nPaper title: Phylogenetic analyses provide the first insights into the evolution of OVATE family proteins in land plants. \\n\\n(#1) The OVATE gene was first identified as an important regulator of fruit shape in tomato, in which a naturally occurring premature stop codon in OVATE results in pear-shaped fruit with longitudinal elongation and neck constriction (Liu et al., 2002) .\\n(#2) This revealed a previously uncharacterized class of regulatory genes in plant development, which encode proteins with a conserved 70 amino acid C-terminal domain.\\n(#3) This domain was designated as the OVATE domain, also known as DUF623 (Domain of Unknown Function 623), and proteins containing this domain were designated OVATE family proteins (OFPs), which are found exclusively in plants (Hackbusch et al., 2005; Wang et al., 2007 Wang et al., , 2011 .'"],"application/vnd.google.colaboratory.intrinsic+json":{"type":"string"}},"metadata":{},"execution_count":14}]},{"cell_type":"code","source":["len(paras_list)"],"metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"JXSwm_pKxOam","executionInfo":{"status":"ok","timestamp":1712916345110,"user_tz":-120,"elapsed":258,"user":{"displayName":"Anna Kiepura","userId":"06512247374108941998"}},"outputId":"2914676b-ebe1-4e11-d5da-7c8955e157af"},"execution_count":15,"outputs":[{"output_type":"execute_result","data":{"text/plain":["981"]},"metadata":{},"execution_count":15}]},{"cell_type":"code","source":[],"metadata":{"id":"yNCotJmXxSqy"},"execution_count":null,"outputs":[]}]}